
# Bio

#####  Transversion
  â†” Transition
point mutation w/ purine(A/G, two ring) â†” pyrimidine(T/C, one ring)

##### Transition

##### Exaptation
= Co-option

##### G-band
= Giemsa banding

##### Blacklist       
- ENCODE Blacklist project. 
- Cut&Run
- 

##### ECM
Extracellular Matrix
ì£¼ë¡œ collagen, gibronectin, laminin, gelatinë“±ì˜ êµ¬ì„±
##### EMT
Epithelial-mesenchymal Transtition
Adherent â†’ Suspension
##### MET
Mesenchymal-epithelial Transtition
Suspension â†’ Adherent
##### MoA
Mechanism of Action
ì£¼ë¡œ ì–´ë–¤ ì•½ë¬¼ì— ëŒ€í•´ ê·¸ ì‘ìš© ê¸°ì „ ì „ì²´ë¥¼ ë§í•¨
##### Cohort

##### FFPE
formalin-fixed paraffin-embed

##### TAD
Topologically Associated Domain
Chromosome ìƒì—ì„œ ì¼ë¶€ chromatinë“¤ì´ 3d foldë¥¼ í†µí•´ ì„œë¡œ interactioní•˜ëŠ” ì˜ì—­

í•˜ìœ„ëŠ” Loop
ìƒìœ„ëŠ” Compartment(Chromosomeì˜ ì¼ë¶€ ì˜ì—­) / Territories(ì—¬ëŸ¬ Chromosomeì´ ì¡´ì¬ ê°€ëŠ¥í•œ nuc. ì˜ ì¼ë¶€ ì˜ì—­)

##### CTCF
CCCTC Binding Factor / Transcriptional repressor
Cohesin loading í›„ Chromatin loop í˜•ì„±


##### MOI
Multiplicity of Infection
Cellë‹¹ ê°ì—¼ë˜ëŠ” ë°•í…Œë¦¬ì•„ì˜ ë°°ìˆ˜

##### CDS
Coding Sequence
ORFì¤‘ splicingë˜ì–´ ì‹¤ì œ ë‹¨ë°±ì§ˆë¡œ ë²ˆì—­ë˜ëŠ” ë¶€ë¶„(exons)

##### UTR
Untranslated Region
3', 5'ì— ORF ë°–, ë²ˆì—­ë˜ì§€ ì•ŠëŠ” ë¶€ë¶„

##### ORF
ORF(Open Reading Frame) : Start codon - Stop codonê¹Œì§€ì˜ ëª¨ë“  ë¶€ë¶„(Intron + Exon)

##### uORF
upstream open reading frame

##### Isoform
Same locus / Transcription - splicing í›„ì˜ mRNA strands

EX) Most eukaryotic genes encode multiple mRNA isoforms

##### G4 Structure
[HÃ¤nsel-Hertsch, R., Di Antonio, M., & Balasubramanian, S. (2017). DNA G-quadruplexes in the human genome: detection, functions and therapeutic potential. _Nature reviews Molecular cell biology_, _18_(5), 279-284.](https://www.nature.com/articles/nrm.2017.3)

##### AAV
Adeno-associated Virus
Used for viral vector delivery

##### SBS
Single Base Subtitution

##### SPG
Spatially Predictive Genes

##### BER
Base Excision Repair.

##### TC-NER
Transcription-Coupled Nucleotide Excision Repair
RNA pol IIê°€ DNA damageë¥¼ ê°ì§€ í›„ ë©ˆì¶œ ë•Œ ë°œìƒí•˜ëŠ” ë³µêµ¬ ê¸°ì‘

##### MMR
Mismatch Repair

##### NHEJ
Non-Homologous End Joining

##### HR
Homologous Recombination
ë³´í†µ DSB ìˆ˜ë¦¬
Compare with sister chromatid, forms D-loop(Holiday junction) while repair process.
https://www.nature.com/articles/cr20081

##### SV
Structual Variation
êµê³¼ì„œì—ì„œ ë´¤ë˜ ì—­ì¢Œ, ì „ìœ„ ë“± chromosome ë‹¨ìœ„ ë³€í˜•ë“¤.
##### IVT
In Vitro Transcript
In vitroë¡œ ë§Œë“  RNA strand\
##### Klenow Fragment
=Large fragment of E.coli DNA polymerase I
3â€™â†’5â€™ Exonuclease
ë³´í†µ Blunt end ë§Œë“¤ ë•Œ ì‚¬ìš©
##### TAESP
Totally asymmetric simple exclusion process
a kind of translation model
##### SNP
Single Nucleotide Polymorpism
gene ìƒì—ì„œ í•˜ë‚˜ì˜ ì—¼ê¸° ë³€ì´
##### Transformation
Non-viral DNA â†’ Bacteria cell
##### Transfection
Non-viral DNA/RNA â†’ Eukaryotic Cell
##### Transdution
Viral DNA/RNA â†’ Bacterial / Eukaryotic Cell
##### Competent Cell
Transfection / Transformation / Transdution ì´ ì˜ ë˜ë„ë¡ ë§Œë“  Cell.

##### Metastasis
ì „ì´

##### Spillover
í˜•ê´‘ ìŠ¤í™íŠ¸ëŸ¼ì—ì„œ ì—¬ëŸ¬ dyeê°„ íŒŒì¥ëŒ€ê°€ ê²¹ì³ì„œ ë°œìƒí•˜ëŠ” ì¸ì‹ ë¬¸ì œ.
ì´í›„ Compensation controlì„ ë”°ë¡œ ì¸¡ì •í•´ì„œ í™•ì¸í•´ ì¤˜ì•¼ í•¨

##### HLA
Human Leukocyte Antigen
ì¡°ì§ ì í•©ì„±ì— ê´€ë ¨ëœ Antigenìœ¼ë¡œ, ì‚¬ëŒì—ì„œ 6ì¢…ë¥˜ ë°œí˜„ì¤‘.
HLA typingìœ¼ë¡œ ì¡°ì§ê°„ ë©´ì—­ ë°˜ì‘ì„ ì¶”ë¡  ê°€ëŠ¥

##### TPRT
Target site primed reverse transcription
non-LTR Transposonì´ encodingí•˜ëŠ” enodnucleaseì— ì˜í•´ ë°œìƒí•˜ëŠ” RT+Priming mechanism

ê³¤ì¶©ì—ì„œ R1, R2ëŠ” 28S rDNA locusë¥¼ íƒ€ê²Ÿìœ¼ë¡œ,
R1ì˜ ê²½ìš° APE(DNA Repair endonuclease)ì™€ ìœ ì‚¬í•œ enodnucleaseë¥¼
R2ì˜ ê²½ìš° type IIS restriction endonucleaseì™€ ìœ ì‚¬í•œ endonucleaseë¥¼ ì½”ë”©í•¨

##### TSD
Target Site Duplication
TPRTê°™ì´ insertionë  ë•Œ, ì–‘ìª½ strandê°€ ì™„ì „ ê°™ì€ ìœ„ì¹˜ì—ì„œ ì˜ë¼ì§€ì§€ ì•ŠìŒ(bluntí•˜ì§€ ì•ŠìŒ)
ê·¸ë˜ì„œ insertion ì´í›„ repair ê³¼ì •ì—ì„œ ì–‘ìª½ì— ê°™ì€ sequence ê°€ ë°˜ë³µë˜ëŠ” íŒ¨í„´ì´ ë‚˜ì˜´

##### H&E staining
H : Hematoxylin + E : Eoisin Staining
í”íˆ ë³´ëŠ” ë¶„í™ìƒ‰-ë³´ë¼ìƒ‰ ì¡°ì§ ì—¼ìƒ‰ ê¸°ë²•.
ì˜í•™/ì¡°ì§í•™ì—ì„œ ë§ì´ ì“°ê³ ,  ê°ê° í•µì´ë‘ ì„¸í¬ì§ˆ ì—¼ìƒ‰ìš©

##### IHC staining
Immunohistochemistry staining
íŠ¹ì • ë‹¨ë°±ì§ˆë§Œ ì¡ì•„ì„œ staining

##### Bromophenol Blue
Protein staining / Gel electrophoresis ì‹œ ì—¼ìƒ‰í• ë•Œë„ ì”€.

##### Coomassie Brilliant Blue
WB ë“±ì—ì„œ protein staining

##### Silver staining
WB ë“±ì—ì„œ protein staining ì‹œ ì‚¬ìš©



##### DAPI 
Aka, `4', 6-diamidino-2-phenylindole`
358nm() - 48?nm() íŒŒë‘ìƒ‰ dye, DNA ì£¼ë¡œ stainingí•¨. (RNA ì•½í•˜ê²Œ?)
í•µ ë³¼ë•Œ ì”€

##### Warburg effect
ë°”ë¥´ë¶€ë¥´í¬ íš¨ê³¼
Tumorgenesis ì´í›„ì—, glucose metabolismì´ glycolysis(>pyruvate, TCA, oxidative phosporelation)ë˜ëŠ”ê²Œ ì•„ë‹ˆë¼ anarobic metablosim(>lactate)


##### q-arm
<> p-arm
chromosome ì—ì„œ centromiere ì•„ë˜ìª½ arm.

##### LNP
Lipid NanoParticle

Phospolipid / polyethylene glycol / cholesterol + (mRNA)


##### CRE
cis-regulatory elements

non-coding DNA ì˜ì—­ ì¤‘, ì£¼ë©´ ìœ ì „ì²´ì˜ ë°œí˜„ì„ ì¡°ì ˆí•˜ëŠ” ìš”ì†Œ

##### FRiP
Fraction of Reads in Peaks

### Clinic

##### SSF
Site-Specific Factor


##### IDC-10
International Statistical Classification of Diseases and Related Health problems
Topography(ì›ë°œë¶€ìœ„) ê¸°ì…/coding ì‹œ ì‚¬ìš©í•˜ëŠ” ë°©ë²•. 
ë‹¤ë§Œ ê·¸ í–‰íƒœ(Behavior)ì™€ ì§„ë‹¨(Morphology) ë”°ë¼ ì½”ë“œ ì„¸ë¶„í™”
`C34.9 ì´ë‚˜ D38.1` ê°™ì€ í˜•íƒœ
##### ICD-O
International Classifications of Diseases of Oncology
Morphology ê¸°ì…/coding ì‹œ ì‚¬ìš©í•˜ëŠ” ë°©ë²•. 
`C42.3, 2563/3` ê°™ì€ í˜•íƒœ
Morphology(ì§„ë‹¨ëª…) 4ìë¦¬ + ê·¸ í˜•íƒœ ì½”ë“œ ë¶™ì–´ìˆìŒ.


##### AJCC Staging
TNM Staging. T(Primary Tumor), N(Regional Lymph nodes), M(Distant Metastsis)ì˜ ì•” ë¶„í¬ ì–‘ìƒì„ í‘œí˜„.
ìˆ«ìê°€ ë†’ì„ ìˆ˜ë¡ ìœ„í—˜í•¨
T0 : Primary Tumorì˜ ê·¼ê±° ì—†ìŒ >> T1s : Carcinoma >> T1, 2 3 4 : Increasing Tumor
N0 : Regional Lymph nodeì˜ metastasis ì—†ìŒ >> N1 2 3 : Regional lymph involve
M0 : No distant metastasis >> M1 : Present distant metastasis

##### SEER Staging
ì•”ê³¼ ì›ë°œë¶€ìœ„ ê°„ ê±°ë¦¬ ê¸°ë°˜ ë¶„ë¥˜.(General staging Summary staging, California staging)
0 : In situ(ì²˜ìŒ ìƒê¸´ ë¶€ìœ„ë§Œ ì¡´ì¬)
1 : Local(ì¥ê¸° ê²½ê³„ ì•ˆìª½)
2+ : Extent

##### RiA
Radio-Immunoassay

##### iRMA
Immunoradiometric assay

https://nm.amc.seoul.kr/asan/depts/nm/K/bbs.do?pageIndex=7&menuId=920&contentId=112150&searchCondition=&searchKeyword=

### Motifs

##### TATA box

##### CpG site
C phosphate G - site
Cytosine can be methylated into 5-methylcytosine, 5-methylcytosine deaminated into thymine

transcription start site located near

If CpG shows high freq, called CpG island.
##### Kozak Consensus sequence
In eukaryotic mRNA, motif of protein translation init.
5â€™ - (gcc)gccRccAUGG - 3â€™

##### Shine-Dargano Sequence
=SD sequence
In prokaryotic mRNA, motif of protein translation init.
5â€™ - GAUCAUUA - 3â€™
##### 5â€™ TOP
5' Terminal oligopyrimidine tract
approx ~12aa seq(pyrimidine) @ 5' m7G cap
##### eQTL
 expression quantitive trait loci
 expression levelì— ì˜í–¥ì„ ì£¼ëŠ” ì—¼ê¸° ë³€ì´ê°€ ìˆëŠ” ê²Œë†ˆ ì˜ì—­

##### PAM sequence
NGG - 3â€™
CRISPR-Cas9 System, tracrRNA detection site

##### DRACH motif
A/G/U - A/G - A - C - A/C/U motif(IUPAC ì½”ë“œë‹¤)
mRNAì—ì„œ m6A ê´€ë ¨í•œ Motif.
## Pathways
https://reactome.org/PathwayBrowser/

### NF-ÎºB

### MAP

### JAK


## Proteins

##### BSA
Bovine Serum Albumin
= Fraction V
ê°’ì‹¸ì„œ ë‹¨ë°±ì§ˆ ë†ë„ ì¡ê±°ë‚˜, ë‹¤ìš©ë„ ì‚¬ìš©â€¦
Blocking bufferì— ì‚¬ìš©. Skim milkë„ blocking
##### HRP
Horse Radish Peroxidase. 
Can oxidize luminol with H2O2, cause chemiluminescent(ABC reaction)
secondary abìš© conjugate enzyme ex)Avidin-HRP(ABC) system
* Buffer ë‚´ [[Terms#Sodium Azaide|NaN3]] ê°€ Inhibitorë¡œ ì‘ìš©í•  ìˆ˜ ìˆìŒ

##### AP
Alkaline phosphatase
secondary abìš© conjugate enzyme
* Buffer ë‚´ Phosphate(ex:PBS)ê°€ Inhibitorë¡œ ì‘ìš©í•  ìˆ˜ ìˆìŒ

##### AFP
alpha-fetoprotein
ê°„ì•” ì§€í‘œ!

##### Trypsin
Serine protease. Argine/Lysineì˜ Cë‹¨ì„ ì ˆë‹¨í•¨
Tryptonì´ë€ ì´ë¦„ìœ¼ë¡œ ì‹œì•½í™”ë¨.
ì£¼ë¡œ dishì— ë¶™ì€ cell(Adherent cells)ì„ ë–¨ì–´íŠ¸ë¦´ ë•Œ ì‚¬ìš©
ë¶€ì°©í•˜ëŠ” AJ(Adherent junction)/TJ(Tight junction) proteinë“¤ì´ ì£¼ë¡œ cofactor Ca2+ë¥¼ ì¨ì„œ EDTAì™€ í•¨ê»˜ ë–¼ì–´ëƒ„
Enzymatic dissociation ë°©ë²•ì´ë¼ stressë¥¼ ì£¼ê¸´ í•˜ì§€ë§Œ, ì˜ ì•ˆ ë–¨ì–´ì§€ë©´ Dispase, Collagenase ë“±ì„ ê°™ì´ ì“°ëŠ” ê²½ìš°ë„ ìˆìŒ.
https://www.thermofisher.com/kr/ko/home/references/gibco-cell-culture-basics/maintaining-cultured-cells.html

##### Miyazawa-Jernigan potential matrix

## Chemical


##### EGTA
Calcium chelator.
Caë¥¼ ì‚¬ìš©í•˜ëŠ” ë‹¨ë°±ì§ˆì„ ì €í•´(No influx-apoptosis)
##### Tris
Tris(hydroxymethyl)aminomethane
NHâ‚‚C(CHâ‚‚OH)â‚ƒ
ë§¤ìš° ê°•í•œ ì—¼ê¸°ì„± = Anion ê³µê¸‰
##### EDTA
+2 Ion chelator. - arm 4ê°œ
ì£¼ë¡œ Mg2+ë¥¼ ì‚¬ìš©í•˜ëŠ” ë‹¨ë°±ì§ˆ(DNAse, RNAse) ì €í•´
 ê·¸ë˜ì„œì¸ì§€ ìš°ë¦¬ ë©ì—ì„  metal ion ì¡ì•„ì„œ oxidative damageë¥¼ ë§‰ì„ ë•Œë„ ì“°ë”ë¼
SDS-PAGEì—ì„œ DNA ì•ˆì •í™”í•˜ëŠ” Mg2+, Ca2+ë“±ì„ ë¹¼ë‚´ DNA (-) charge ì™€ ë¶„ìëŸ‰ correl
ì‹ë¬¼ ì„¸í¬ì˜ ê²½ìš° ì„¸í¬ë²½ì˜ Ca2+ë¥¼ ë¹¼ë‚´ì„œ ì„¸í¬ë²½ íŒŒê´´ì—ë„ ì‚¬ìš©
Dissociation ì‹œ trypsinê³¼ í•¨ê»˜  adherent proteinì´ ì“°ëŠ” Ca2+ë¥¼ í›”ì³ê°€ì„œ ê²°í•©ì„ ì•½í•˜ê²Œ í•´ì¤Œ.
##### EGTA
Aminopolycarboxylic acid = egtazic acid
Mg ì¹œí™”ë ¥ì´ ë‚®ì•„ì„œ Caë¥¼ ë” ì˜ ë‹¹ê¹€
##### NP-40
Mild Non-ionic Detergent.
ì„¸í¬ë§‰/í•µë§‰ íŒŒê´´ì‹œ ì‚¬ìš©. TritonX-100, Tween20ì´ ê°™ì€ ì—­í• 

##### Digitonin
MILD detergent
ATAC-seq ë“±ì—ì„œ ì•½í•˜ê²Œ êµ¬ë© ëš«ì„ ë•Œ ì‚¬ìš©. Tween20/NP40ë„ ì”€
##### TritonX
Mild Non-ionic Detergent
##### Tween20
Mild Non-ionic Detergent



##### DFAM
Deforoxamine mesylate
kind of antioxidant

##### BHT
Butylated hydroxytoluene
kind of antioxidant

##### ARP
=Aldehyde Reactive Probe ~~Adress resolution protocol~~
Aldehydeë‘ keteonê³¼ ë°˜ì‘í•´ biotinylation 
##### NaF
Phosphatase/ Serin protease Inhibitor
##### Sodium Azaide
=NaN3
ë°©ë¶€ì œ/í•­ì§„ê· ì œ
ë²„í¼ ì˜¤ë˜ ë³´ê´€í•˜ê³  ì‹¶ì„ ë•Œ ì“°ì§€ë§Œâ€¦ ì‚¬ëŒë§ˆë‹¤ ì˜ê²¬ì´ ê°ˆë¦¼.
Leadë‚˜ Copper (Plumbing)ì— ë°˜ì‘í•´ ê¸ˆì† ì§ˆì‚°ì—¼ìœ¼ë¡œ í­ë°œì„±ì„ ëŒ ìˆ˜ ìˆìœ¼ë‹ˆ ì£¼ì˜.
##### Chaotropic agent
Hydrogen bonding ì œê±°ìš© ë¬¼ì§ˆ(ì£¼ë¡œ 1ê°€ ì—¼)
ex) Guanidin sodium salt, urea, sodium iodide
Protein unfolding/denature, Cell lysis, DNA-silca ion exchange bridge ë“±

##### DEAE
Diethylaminoethyl celluose
pl ìœ„ì˜ pHì—ì„œëŠ” ì–‘ì „í•˜ë¥¼ í†µí•´ ìŒì „í•˜ ë‹¨ë°±ì§ˆ ë“±ì„ ì •ì œì‹œ ì‚¬ìš©(sepharose)
ì´í›„ pHë‚˜ ì—¼ë„ ë³€í™”ë¡œ ì •ì œ

##### Sepharose
Agarë¡œ ë§Œë“  bead
IP ë“±ì—ì„œ Affinity bead, chromatography ë“±ì— ì‚¬ìš©


##### Streptavidin
=~Avidin
ì›ë˜ Avidinì€ ë‹¹ë‹¨ë°±ì§ˆë¡œ, 4 subunit <> 4 biotin attract, ë‹¹ì´ ë‹¨ë°±ì§ˆì˜ 10%ì •ë„.
ê·¼ë° Streptavidinì€ ë‹¹ ì—†ì´, 4 subunit, less non-specefic interaction, ë‚®ì€ í•´ë¦¬ë„
ë¬¸ì œëŠ” RYD(Arg-Tyr-Asp) ì„œì—´ì´ cell adhesion motifì¸ RGDì™€ ìœ ì‚¬í•´ì„œ cell surfaceì— ë¹„íŠ¹ì´ì ìœ¼ë¡œ ë¶™ì„ ìˆ˜ ìˆëŠ”ë°, ì´ëŠ” ì¬ì¡°í•©ëœ Neutravidin/Extravidin ì‚¬ìš©í•˜ë©´ ë¨.


##### ABC
Avidin-Biotin complex

##### Protease/Phosphatase inhibitor cocktail
Leupeptin (10mg/ml)
Aprotinin (10mg/ml)
PMSF (100mM)
Okadaic acid (20nM)

##### Heparin
RNAse inhibitor.
í˜ˆì•¡ ì‘ê³  ë°©ì§€ ë¬¼ì§ˆë¡œë„ ì•Œë ¤ì§
##### Triton
Cell lysis, Protein precipitation
##### DMSO
Dimethyl Sulfoxide
Cell stockì´ë‚˜ cultureì—ì„œëŠ” ì–¼ë¦° cellì´ í„°ì§€ì§€ ì•Šë„ë¡ ë°©ì§€
Freezing mediaì— ë³´í†µ 10%ë¡œ ë“¤ì–´ê°
ì‹¤ì˜¨ì—ì„œëŠ” cell membraneì„ íŒŒê´´í•˜ê¸° ë•Œë¬¸ì— ì£¼ì˜

ì‚¬ìš© ì‹œ ì–¼ë¦´ ë•Œì—ëŠ” cryopreservation freezing containerì— ë„£ê³  LNì—ì„œ ë‘ë©´ ë¶„ë‹¹ 1ë„ì”© ì²œì²œíˆ
ë‹¤ì‹œ í•´ë™ ì‹œì—ëŠ” 37ë„ bathì—ì„œ ë¹ ë¥´ê²Œ ë…¹ì´ê³  DMSOë¥¼ .5% ì´í•˜ë¡œ í¬ì„í•´ ì¤˜ì•¼ í•¨.
##### FBS
Fetal Bovine Serum
Cell cultureì‹œ ì‚¬ìš©. ë‹¤ë§Œ complementë‚˜ inhibitorë“±ì´ ì¡´ì¬í•´ ë©´ì—­ ì„¸í¬ ë°°ì–‘ ì‹œ ABì™€ ê²°í•©-cytotoxictyë¥¼ ë³´ì¼ ìˆ˜ ìˆìœ¼ë¯€ë¡œ ES cell, insect cell lineë“±ì—ì„œëŠ”  heat inactivationí•´ ì¤˜ì•¼ í•¨.
[[Terms#Trypsin |Trypsin]] inhibitor(alpha-2-macroglobulin)ì´ ì²¨ê°€ë˜ì–´ ìˆì–´ì„œ ì´í›„ Trypsinì²˜ë¦¬ ì‹œì—ëŠ” DBPSë“±ìœ¼ë¡œ washí•´ì¤˜ì•¼ í•¨. ë°˜ëŒ€ë¡œ ì¬ë°°ì–‘ì‹œ FBS ì²˜ë¦¬ í•´ì£¼ë©´ trypsinì´ ë‹¤ì‹œ ì •ì§€.

##### alpha-2-macroglobulin
Trypsin inhibitor.




##### TRIzol
=TRI Reagent
DNA/RNA/Protein Isolation ê³¼ì •ì—ì„œ ì‚¬ìš©ë˜ëŠ” ì‹œì•½â€¦
ê¸°ì¡´ LiCl Precipitation(Phenol-Ethanol+ LiCl(Lithium Chloride) + Ethanol) ê¸°ë°˜ ê¸°ë²•ì—ì„œ Trizolì˜ Guanidine isothiocyanateê°€ RNAse ì–µì œ.
ë‹¤ë§Œ ìƒëŒ€ì ìœ¼ë¡œ DNA contamination ê°€ëŠ¥ì„±ì´ ìˆìŒ
ë°©ë²•ì€ ë¹„ìŠ·! 
ì‹«ìœ¼ë©´ column ì¨ì•¼ì§€ ë­â€¦
##### ME
=beta-mercaptoethanol, $\beta$ ME
Disulfide bond ì œê±°, -OH radical ì œê±°
ì£¼ë¡œ í•­ì‚°í™”ì œ ì—­í• ë¡œ DNA ë“± ë³´ê´€ ì‹œ ì‚¬ìš©í•˜ê±°ë‚˜
RNAseì˜ Disulfide bond ë¶€ìˆ´ RNA isolation ì‹œ íŒŒê´´ë˜ì§€ ì•Šë„ë¡ ì‚¬ìš©

##### Chloroform
~~Itâ€™s chlorolform~~
RNA prepì—ì„œëŠ” Pheonl ì œê±°

##### Isopropanol
RNA prepì—ì„œëŠ” RNA precipitation(Illusion)

##### DFOM
= DFO, DFOA, Desferroxamine
Reduce In-vivo oxidation by chelating 2/3 ions (like EDTA, inhibiting in-vitro fenton)

##### DTT
Dithiothreitol
Redox, Reduce disulfide bonds, Thiolated DNA reducer

##### Cycloheximide
Translation / Ribosome blocker

##### DPEC
Diethyl pyrocarbonate
RNase inhibitor

##### GST
GST elution & Reduced Glutathione elution buffer
GST-tagëœ ë‹¨ë°±ì§ˆì€ í•´ë‹¹ bufferì—ì„œ elution, preserve
##### Isopropanol
ì„¸í¬ ë‹¨ë°±ì§ˆê³¼ DNA ë¶„ë¦¬, DNA ì¹¨ì „

##### PCI
Phenol:Chloroform:Isoamyl alcohol (25:24:1)

Phenol : dissolve protein
Chloroform : dissolve hydrophobics/Phenols
Isoamyl alcohol : Protect phenol from oxygen, dissolve hydrophillics

Centrifugeí•˜ë©´ ì•„ë˜ì— Chroloform-phenol ì¸µ, ì¤‘ê°„ì— ë‹¨ë°±ì§ˆ, ìœ„ì— Isoamyl alcohol 

##### DPBS
Dulbeccoâ€™s Phosphate buffered saline
PBSì²˜ëŸ¼ bufferì´ê³ , ì—¬ê¸°ì— Mg2+ ë‚˜ Ca2+(MgCl2, CaCl2) ê°™ì€ ì„¸í¬ ìƒì¥(ë§‰ í‰í˜•ì´ë‚˜ ì‹ í˜¸, íš¨ì†Œ ë“±) ì— í•„ìš”í•œ ì´ì˜¨ì´ í¬í•¨ë¨
ì¶”ê°€ë¡œ Glucoseê°€ ë“¤ì–´ê°€ëŠ” ê²½ìš°ë„ ìˆìŒ.
### Dyes / Indicators

https://www.bio-rad-antibodies.com/spectraviewer.html
##### FITC
Fluorescein isothiocyanate
##### CFSE
ì„¸í¬ë§‰ì„ íˆ¬ê³¼í•´ì„œ ë‹¨ë°±ì§ˆì— ê²°í•©í•˜ëŠ” dye. (Parish, 1999)
ë³´í†µ cell proliferationë§ˆë‹¤ ì´ë¡ ìƒ $1/2^n$ê¼´ë¡œ ê°ì†Œí•  ê²ƒì´ê¸° ë•Œë¬¸ì— ë¶„ì—´ ì‹œê¸°ë¡œ êµ¬ë¶„ ê°€ëŠ¥.

##### PE
Phycoerythrin

##### APC
Allophycocyanin

##### Sybr dye

##### Annexin V
Phospholipid ì¤‘ Phosphatidylserine(PS)ì™€ ê²°í•©í•˜ëŠ” dye
Apoptosis ì‹œ PSê°€ ì„¸í¬ ë°”ê¹¥ìœ¼ë¡œ ì¬ë°°ì¹˜ë˜ë¯€ë¡œ, ì´ë¥¼ ê°ì§€í•  ìˆ˜ ìˆìŒ
Ca2+ í¬í•¨ëœ buffer ì‚¬ìš©í•´ì•¼í•¨

##### Tandem Dye
ë‘ í˜•ê´‘ë¬¼ì§ˆì´ ë¶™ì–´ ìˆëŠ” FRETê³¼ ë¹„ìŠ·í•œ ì‘ìš©í•˜ëŠ” Dye.
Batch/Vialë§ˆë‹¤ ê²°í•©ë¹„ë‚˜ ë†ë„ ì°¨ì´ê°€ ë°œìƒí•  ìˆ˜ ìˆì–´ì„œ ì£¼ì˜.
ex) PE-Cy7, PE-eFluor 610, PerCP-Cy5.5, APC-Cy7


##### AF series
Alexa Fluor Dye. AF N ê¼´ì´ë©° Exciation wavelengthê°€ ê¸°ë¡ë˜ì–´ ìˆìŒ
https://www.bdbiosciences.com/en-us/applications/research-applications/multicolor-flow-cytometry/product-selection-tools/spectrum-viewer

##### eFluor series
eFluor N ê¼´ì´ë©° Emission wavelenghê°€ ì í˜€ ìˆìŒ.
https://www.thermofisher.com/us/en/home/life-science/cell-analysis/labeling-chemistry/fluorescence-spectraviewer.html#!/

##### GECIs
Genetically encoded calcium indicators.
Calmodulinì´ ëŒ€í‘œì 


### Antibiotics
P/S : Penicillin/Streptomycin. P/S double tolerent bacteriaë„ ì¡´ì¬

##### Penicillin
ê·¼-ë³¸ antibiotics. Bacteria ëŒ€ìƒ
##### Sterptomycin
ì—­ì‹œ ì„¸ê· ìš© í•­ìƒì œ.

##### Gentamicin
ì„¸ê· ìš© í•­ìƒì œ

##### Amphotericin B
Antimycotic(ì§„ê· ìš© í•­ìƒì œ)

##### Ampicillin

##### Carbenicillin

##### Chloramphenicol

##### Kanamycin

##### Tetracycline

##### Neomycin
Bind prokaryote ribosome, block polypeptide 

##### G418
~Gentamycin B1
Block Polypeptide 

### Buffers

##### TAE
TAE buffer
Tris + Acetate + [[Terms#EDTA |EDTA]]
Trisê°€ ì–‘ì´ì˜¨ ê³µê¸‰, Acetateê°€ pH ì¡°ì ˆí•˜ëŠ” buffer

ì£¼ë¡œ ì „ê¸°ì˜ë™ì—ì„œ ì‚¬ìš©
##### TBE 
TBE buffer
Tris + Borate + [[Terms#EDTA |EDTA]]
Boric Acidë¡œ ì¡°ì ˆí•œ ê²½ìš°. TAEë³´ë‹¤ ë¹„ì‹¸ì§€ë§Œ ë†’ì€ gel electorphoresis resolution, capacity. (ì£¼ë¡œ ì‘ì€ í¬ê¸° DNA í™•ì¸ ì‹œ)
ë‹¤ë§Œ ê·¸ë˜ì„œ ì´ë™ì´ ëŠë¦¼

ì£¼ë¡œ ì „ê¸°ì˜ë™ì—ì„œ ì‚¬ìš©

##### SB
Sodium Borate buffer
TAE, TBEë³´ë‹¤ ë¹ ë¥´ì§€ë§Œ íš¨ìœ¨ ì•ˆì¢‹ì•„ì„œ(+ì—´ ë” ë‚œë‹¤ê³  í•¨) ì˜ ì•ˆì”€..

##### Sodium Acetate buffer

##### Washing buffer
ì£¼ë¡œ EtOH, DNAë¥¼ ì œì™¸í•œ salt ë“±ì„ wash
##### Elution buffer
Silica resin columnìœ¼ë¡œë¶€í„° DNAë¥¼ ë…¹ì—¬ì„œ ë–¼ì–´ëƒ„
##### Phosphate buffer

##### PBS
Phosphate Buffered Saline
Buffer + Osmolarity ë‹¤ ì±™ê¸´ ê±°ì´ ê¸°ë³¸ ë²„í¼.
Cell dislocation, immunostaining, cytometry ë“±ë“±â€¦
###### DPBS
PBS + potassium chloride(KCL). (+ca, mg, glu, pyruvate)
ë¹„ìŠ·í•œë°, ì£¼ë¡œ tissueì— ì•ˆ ì¢‹ì€ ì˜í–¥ì„ ì¤„ê±° ê°™ì„ ë•Œ(Dissection, perfusion, IP, cell therapy) ì¡°ê¸ˆ ë” ì„ í˜¸
KCLì´ ì„¸í¬ ë…ì„±ì„ ë‚®ì¶°ì¤€ë‹¤ê³  í•¨.(Na/K ATPASE pump?)
##### PBE
Phosphate Buffered [[Terms#EDTA |EDTA]]
##### Tris-HCL
acidic Buffer. 
##### TBS buffer
Tris-buffered saline buffer

##### PBST buffer

##### TBST buffer
##### RIPA buffer
containing ionic detergent, ion chelator + (protease inhibitor) with Tris-HCL buffer
detergentë¡œ cell/nuclear membraneì„ ê¹¨ê³  EDTA+inhibitorë¡œ protein ë³´í˜¸
ì£¼ë¡œ Western blotì—ì„œ ë‹¨ë°±ì§ˆ ì¶”ì¶œ ì‹œ ì‚¬ìš©
##### Laemmli buffer

##### HEPES buffer
HEPES(N-2-hydroxyethylpiperazine-N'-2-ethanesulfonic acid)
- Useful at long-time incubation?
- Show cytotoxicity (reaction with light, create h2o2)
##### EBSS
Earleâ€™s Balanced Salt Solution
ë‘˜ ë‹¤ osmolarity + PH ì±™ê¸°ëŠ” culture ìš© ë²„í¼ëŠ” ë§ìŒ.(PBS + alpha ëŠë‚Œ)
ì£¼ë¡œ glucose, pyruvate, Ca, Mg ì¶”ê°€ë¨.

ë‹¤ë§Œ Co2 í‰í˜•ì„ sodium bicarbonate(HCO3-)ë¡œ ìœ ì§€í•´ì„œ Co2 incubatorê°€ í•„ìš”.
##### HBSS
Hankâ€™s Balanced Salt Solution
ë‘˜ ë‹¤ osmolarity + PH ì±™ê¸°ëŠ” culture ìš© ë²„í¼ëŠ” ë§ìŒ.(PBS + alpha ëŠë‚Œ)
ì£¼ë¡œ glucose, pyruvate, Ca, Mg ì¶”ê°€ë¨.

EBSSë³´ë‹¤ sodium bicarbonateê°€ ëœ ë“¤ì–´ê°€ì„œ Co2ê°€ ì ì€ ì¼ë°˜ ëŒ€ê¸°ì—ì„œ í‚¤ìš¸ ìˆ˜ ìˆìŒ.
##### Ealreâ€™s Salt

##### Hankâ€™s Salt



### Media
aka, Broths
##### LB media
Lysogeny Broth / Luria-Bertani Broth
10g Tryptone
.5g Yeast extract
10g NaCl
in liter
Lennox mediaì˜ ê²½ìš° 5g NaCl
Miller mediaëŠ” ìœ„ ì¡°ì„±ê³¼ ê°™ìŒ
##### TB media
Terrific Broth
11.8g Tryptone 
23.6g Yeast extract
9.4g Dipotassium hydrogen phosphate
2.2g Potassium dihydrogen phosphate
4ml .4% Glycerol
in liter
neutrient-rich, glycerolì„ ì¤˜ì„œ glucose ê³„ì—´ ëŒ€ì‚¬ê°€ ì–µì œë¨(lacI)
##### SOC media
Super Optimal broth with Catabolite repression
2g Tryptone
.5g Yeast extract
2ml 5M NaCl
250ul 1M KCl
1ml 1M MgCl2
1ml 1M MgSO4
2ml 1M Glucose
Transformation ì´í›„ cell recoveryì— ì‚¬ìš©
Glucoseë¥¼ ë„£ì–´ì„œ ê¸°íƒ€ catabolic path ì–µì œ
##### BHI media
Brain Heart Infusion
Infusion from brain heart 6g
Peptic digest of tissue 6g
Pancreatic digest of gelatin 14.5g
Sodium chloride 5g
Dextrose(D-Glucose only) 3g
Disodium Phosphate 2.5g
(+ Hemin 5mg)
(+ Vitamin K .1mg)
per liter

ê¹Œë‹¤ë¡œìš´ ë¯¸ìƒë¬¼(Fastidious microorganism)ì— ì“°ì„.
anarobicâ€¦ ë“±
##### CBA media
Columbia Blood Agar
Pancreatic digest of casein 12g
Peptic digest of tissue 5g
Yeast extract 3g
Beef extract 3g
Corn starch 1g
Sodium chrolide 5g
Agar 13.5g
+Sheep blood(Defibrinated) 5% (After agar media, autoclave, cooldown)
ìœ„ì™€ ë¹„ìŠ·í•œ ì—­í• 

##### BME
Basal Media Eagle
ì˜›ë‚  1950ì— Hela cell í‚¤ìš¸ë•Œ ì“´ ë°°ì§€.
ì•„ë¯¸ë…¸ì‚°, ë¹„íƒ€ë¯¼ ì¡°ì„± ê´€ë ¨í•´ì„œ ê¸°ë°˜.
ìš”ìƒˆëŠ” MEMì´ë‚˜ DMDMì„ ì“´ë‹¤â€¦

##### EMEM
Eagleâ€™s Minimal Essential Medium
= MEM
BMEë³´ë‹¤ ë” ë§ì€ ì•„ë¯¸ë…¸ì‚° + EBSS buffer

EBSS base
2mM L-glutamine
1mM sodium pyruvate
1500mg/L sidoum bicarbonate

##### DMEM
Dulbeccoâ€™s Modified Eagleâ€™s Medium
MEM + ì•„ë¯¸ë…¸ì‚°, ë¹„íƒ€ë¯¼, íƒ„ìˆ˜í™”ë¬¼ ë“± ì¶”ê°€
Sodium bicarbonite buffer ì”€ == 5-10% CO2 ë†ë„ í•„ìš”

EBSS base
4mM L-glutamine
4500mg/L glucose
1mM sodium pyruvate
1500mg/L sodium bicarbonate

## Experiments

### Euiqipment

##### E-Tube
eppendorf tube
##### Cryo tube
Cryopreservationìš© Tube.
[[Terms#DMSO]] ì°¸ì¡°!

##### Amber tube
For Photosensitive reagents

##### Collection tube

##### Concical tube
##### Column

##### Cuvette
íë²³. ë³´í†µ ë¶„ê´‘ë„ê³„(Spectrometer)ë¡œ ì½ì„ ë•Œ ì‚¬ìš©
##### AFA Vessels
ë³´í†µ crimp-capped
##### Nanodrop
ë¶„ê´‘ë„ê³„ì˜ˆìš”
##### TapeStation
Electorphoresisë³´ë‹¤ ì¢‹ë‹¤!
##### Capillary Electrophoresis
ì „ê¸°ì˜ë™ì„ ë‘ bufferë¥¼ ì‡ëŠ” ì–‡ì€ capillaryë¥¼ í†µí•´ ì§„í–‰
ê³ ì••ì„ ì‹œê°„ì— ë”°ë¼ ì£¼ë©´ì„œ, ì´ë™í•˜ëŠ” ë¬¼ì§ˆì˜ í˜•ê´‘ë„ë¥¼ ì‹œê°„ì— ë”°ë¼ ì¸¡ì • ê°€ëŠ¥

### Short-term
##### OD
Optical density
##### RFU
Relative Flourence Unit
##### Flaming

##### Resuspend
Centrifuge ë“±ìœ¼ë¡œ ì•‰íŒ(pellet ë“±ìœ¼ë¡œ) cellì„ ë‹¤ì‹œ Media/Mediumìœ¼ë¡œ ì„ì–´ ì£¼ëŠ” ì¼
##### Supernatant
<> Precipitate / Pellet
ìƒì¸µì•¡. 
ë¬´ì¡°ê±´ ë²„ë¦¬ëŠ” ê²ƒë„ ì•„ë‹ˆê³ , RNA prep ì¤‘ì—ëŠ” ì‚¬ì´ ë§‰ì— ì‹ ê²½ ì¨ì•¼ í•  ë•Œë„ ìˆìŒ.
##### Invert
=Flip
##### Vortex

##### Thawing
í•´ë™
37ë„ ë¬¼ì—ì„œ ë¹ ë¥´ê²Œ ë…¹ì´ëŠ”ê²Œ ì¤‘ìš”
##### Aspirate
Suction / í˜¹ì€ plateë¥¼ í„¸ì–´ì„œ media ì œê±°í•˜ëŠ” ê³¼ì •
##### Precipitation
ì¹¨ì „
##### Titration
ì ì •
##### Sucrose gradient
Sucrose cushion.
Sucrose ë†ë„ë¥¼ ë‹¤ë¥´ê²Œ ë§Œë“  ë’¤ ultracentrifuge, gradientëŠ” ì„œì„œíˆ ë¬½ì–´ì§€ëŠ” ë°©í–¥(ìœ„)
cushionì€ ê°€ì¥ ì•„ë˜ íŠ¹ì • ë†ë„ì˜ sucrose ìš©ì•¡ë§Œì„ ê¹”ê³  centrifuge

##### UV ligation

##### Gel Electrophoresis

Agarose ê¸°ì¤€

| Gel Concentration(%) | DNA size(Kb) |
| -------------------- | ------------ |
| .5                   | 1-30         |
| .75                  | .8-12        |
| 1                    | .5-10        |
| 1.5                  | .2-3         |
| 2                    | .01-.5       |

Polyacrylamide gelì˜ ê²½ìš° â†’ PAGE

##### Gel Purification
í•˜ê¸° ì „ long-wavelength UVë¡œ ë³´ê¸°.(DNA íŒŒê´´ ë°©ì§€)


##### Fenton reaction
H2O2 + Fe(II)
ì² ì„ ì´‰ë§¤ë¡œ ì‚°í™”

##### Avidin-Biotin Interaction
=Streptavidin-biotin binding

##### FISH
Fluorescent in situ hybridization

##### Electroporation
= Electropermeabilization


##### Pipetting
https://www.ibric.org/bric/trend/bio-series.do?mode=series_view&newsArticleNo=8858877&articleNo=8882772&beforeMode=series_list&article.offset=0&articleLimit=10#!/list

###### Reverse Pipetting

###### Pre-wetting

### Big-term

##### Northern blot(RNA blot)

##### Western blot(Protein Immunoblot)
= WB, Immunoblot
###### Far-western blot
Complex ê¸°ë°˜ Western blot.
 
##### Southern blot(DNA blot)

##### Eastern blot

###### Far-estern blot
##### PCR
taq polymerase, PFU ë“±ìœ¼ë¡œâ€¦
PrimerëŠ” ë³´í†µ 100uM
T_mì€ primer sequence ë”°ë¼ì„œ, pol step ì˜¨ë„/ì‹œê°„ì€ enzymeì— ë”°ë¼ì„œ

###### Nested PCR
d

###### RACE-PCR
Rapid amplification of cDNA ends - PCR


##### Native-PAGE
Polyacrylamide gel electorphoresis

`%T = Acrylamide + Bisacrylamide concentration`
`%C = Bisacrylamide(cross-linker) concentration related to %T`
%TëŠ” Gelì˜ ë°€ë„ë¥¼, %CëŠ” ê·¸ ì‚¬ì´ì— ë‚œ êµ¬ë©ì˜ í¬ê¸°ë¥¼ ê²°ì •
ë‹¨ë°±ì§ˆ ì‚¬ìš©ì‹œ 29:1(3.3%C), Peptideë¼ë©´ 19:1(5%C)
ë‘ ì¬ë£Œ ì¤‘í•©ìœ¼ë¡œ ì ¤ì„ ë§Œë“¤ê¸° ë•Œë¬¸ì— APS(Ammonium persulfate), TEMEDë¥¼ ì‚¬ìš©í•´ ì¤˜ì•¼ í•¨. 

| Gel Concentration(%T) | Protein size(kDa) |
| --------------------- | ----------------- |
| 20                    | 4-40              |
| 15                    | 12-45             |
| 12.5                  | 10-70             |
| 10                    | 15-100            |
| 8                     | 25-100            |

sample bufferì—ëŠ” reducing agent(2-mercaptoethanol 5-20mM, DTT .5-1mM) ì‚¬ìš©
https://www.ibric.org/bric/trend/bio-series.do?mode=series_view&newsArticleNo=8858225&articleNo=8882788&beforeMode=series_list&article.offset=0&articleLimit=10#!/list
##### SDS-PAGE
Sodium Dodecyl Sulfate - PolyAcrylamide Gel Electrophoresis

##### BLI
BioLayer Interferometry
- ì–‡ì€ ì„¸í¬ë§‰ì— ëŒ€í•´ ì ì™¸ì„ ìœ¼ë¡œ êµ¬ì¡°/ìƒí˜¸ì‘ìš© ì¶”ì •
##### FISH


##### Plaque assay
Virus Titor(Titration)ì— ì‚¬ìš©ë˜ëŠ” ë°©ë²• ì¤‘ í•˜ë‚˜.


##### Blue-White screening
LacZ - Xgal ê¸°ë°˜ì˜ ë²¡í„° ì‚½ì… ìŠ¤í¬ë¦¬ë‹


##### Transfer
ë³´í†µ Wetsern blotì—ì„œ gelâ†’membraneìœ¼ë¡œ ë‹¨ë°±ì§ˆ ì „ì†¡ì‹œ ì‚¬ìš©í•˜ëŠ” ìš©ì–´
Wet
Semi-dry


##### Flow Cytometry
Cellì„ í˜•ê´‘ì„ í†µí•´ ë¶„ë¦¬í•˜ëŠ” ê¸°ë²•.

FSC(Forward Scatter), SSC(Side Scatter)ë¡œ ì„¸í¬ì˜ í¬ê¸°ì™€ granularityë¥¼ gating
ì´í›„ Viability dye(eFluor 506 ë“±)ë¡œ ì£½ì€ cellì„ ê±¸ëŸ¬ëƒ„.


https://www.ibric.org/bric/trend/bio-series.do?mode=series_view&newsArticleNo=8854159&articleNo=8882772&article.offset=0&articleLimit=10&beforeMode=series_list#!/list


###### FSC
Fowrard scatter.
FSC-H, W, Aë¡œ ê°ê° Height, Width, Areaë¥¼ êµ¬ë¶„í•˜ë©° Aì™€ Hê°’ì´ y=x ë°–ì´ë¼ë©´ doublet



##### FACS
Fluorescence Activated Cell Sorting
Flow cytometryëŠ” êµ¬ë¶„/í•„í„°ë§, FACSëŠ” ë¶„ë¥˜ê°€ ê°€ëŠ¥
ì„¸í¬ê°€ ìˆëŠ” dropletì— ì „í•˜ë¥¼ ë¶€ì—¬í•´ì„œ ë‹¹ê¸¸ ìˆ˜ ìˆìŒ
ë¹„ì‹¸ì„œ ëª¨ë¥¸ë‹¤â€¦.

##### PAL-seq
PolyA tail ê¸¸ì´
##### TAIL-seq

##### DMS-seq
=Structure-seq

##### RPF-seq

##### Ribo-seq
Ribosome profiling
mRNAì— ë¶™ì€ Ribosome ìˆ˜ ë˜í•œ expressionì— ì—°ê´€.

##### IP
Immunoprecipitation

Anti Rat â†’ protein A is preferred
Anti Mouse â†’ protein G 

###### DRIP-seq
DNA-RNA hybridì¸ R-loopì„ íƒ€ê²Ÿìœ¼ë¡œ í•˜ëŠ” seq
##### ChIP-seq
Chromatin Immunoprecipitation followed by sequencing
DNA binding protein + í•´ë‹¹ ìœ„ì¹˜ ì°¾ê¸°
IPë¡œ í•´ë‹¹ ë‹¨ë°±ì§ˆë§Œ ë–¨ê¶ˆì„œ sequendcing

###### RIP-seq
RNA Immunoprecipitation ChIP sequencing
##### GRO-seq
Global Run-on sequencing
Nascent(newly synthized)  RNA-transcript seq
~ with BRIC-seq, Bru-seq, BruChase-seq

##### CLIP
Cross-linking and Immunoprecipitation
íŠ¹ì • ë‹¨ë°±ì§ˆì´ ê²°í•©í•˜ëŠ” seq ìœ„ì¹˜ ì°¾ê¸°
crosslinking í›„ IPë¡œ ì°¾ëŠ”ë°, ë¬¸ì œëŠ” í•´ë‹¹ ìœ„ì¹˜ ì„œì—´() ë§Œ ë³¼ ìˆ˜ ìˆì—ˆìŒ

More @ this [paper](https://www.nature.com/articles/s43586-021-00018-1)
###### HITS-CLIP

###### PAR-CLIP

###### iCLIP

###### Clear-CLIP
ì›ë˜ CLIP ì‹œ ìœ„ì¹˜ë§Œ ì•Œê³ , ì–´ëŠ ì„œì—´(í˜¹ì€ RNA)ì¸ì§€ëŠ” ì•Œ ìˆ˜ ì—†ëŠ”ë°(ë¹„ì—°ê²° ë¶€ìœ„ëŠ” degradationë˜ê¸° ë•Œë¬¸)
ì•„ì˜ˆ proteinì´ ì¡ì€ ì„œì—´ì„ í†µì§¸ë¡œ ë“¤ê³  ì˜¤ëŠ” ë°©ë²•
##### irCLIP

###### eCLIP

##### ELISA
Enzume-Linked Immunosorbent Assay
ë‹¨ë°±ì§ˆ ë°œí˜„ëŸ‰ì„ ë³´ê¸° ìœ„í•œ ë°©ë²•(like WB)


##### SPRI
Solid Phase Reversible Immobilization

##### Target-seq
íŠ¹ì • target ì˜ì—­ì„ ì„ íƒ í›„, ì—¬ëŸ¬ ë²ˆì˜ PCR í†µí•´ì„œ ê±¸ëŸ¬ëƒ„

##### LiCl precipitation
RNA Purification & Precipitation
##### Gibson Assembly

##### IP-MS
Immunoprecipation-Mass spectography
##### Trizol Extraction

##### EMSA
Electrophoretic Mobility Shift Assay

##### DCFDA Assay
= H2DCFDA Assay
ROS detection


##### RNA-seq

##### Insert-seq

##### ATAC-seq
Find Open chromatin!
Open chromatinì— Tn5ê°€ íŠ¹ì • adaptorë¥¼ transposition(adaptor insertion)í•¨
ê·¸ëŸ¬ë©´ seqí• ë•Œ ë§ì´ insertion í•œ ì˜ì—­ì´ ê·¸ë§Œí¼ ì½í˜€ì„œ, Tn5 transposition rate = read N = (Promotor accessibliility)

##### 3C methods
Intermolecular range, ì–´ëŠ Chromatinë¼ë¦¬ interactioní•˜ê³  ìˆë‚˜?
ë³´í†µ ë‹¤ë¥¸ chromatinë¼ë¦¬ Crosslink í›„ digestioní•´ì„œ ë¶™ì€ fragmentë¥¼ ì–»ìŒ
###### 3C
qPCRí•´ì„œ framentë¥¼ ligation í›„ ì°¾ê¸°

###### 4C
Second-digestion í›„ self-circulaization, inverse PCRë¡œ seq

###### 5C
ì¶”ê°€ë¡œ Oligoë¥¼ ë¶™ì¸ ë’¤ì— PCR

###### ChIA-PET
Chromatin precipitation í›„ linker ligation, proximity ligation í›„ digestioní•˜ë©´ non-chimeric PETì„ ì¡ì•„ì„œ seq

###### Hi-C
Restriction Enzyme(HindIII)ë¡œ ìë¥¸ ë’¤ biotin marking, ligation í›„ biotin pulldown

##### ChIC
Chromatin Immunoclevage
ê¸°ì¡´ [[Terms#ChIP-seq|ChIP]] ì€ ë§ì€ ì„¸í¬ì™€ ë†’ì€ depth,ë¥¼ í•„ìš”ë¡œ í•¨. ë³µì¡í•˜ê¸°ë„ í•˜ê³ . Immunoprecipitationì´ ì–´ì©” ìˆ˜ ì—†ìŒâ€¦
Antibody sensitivityê°€ ëŠ˜ë©´ì„œ ChICê°€ ê°€ëŠ¥!

###### Cut-and-tag
Cleavage Under Targets & Tagmentation
ì£¼ë¡œ íˆìŠ¤í†¤ PTMê³¼ CTCF(CCCTC-binding factor) mapping

Ab + Secondary Ab í†µí•´ Tn5ë¡œ adaptor ligation ì´í›„ sequencing

Mg2+ë¡œ Tn5 activiation, tagging

	í•œë²ˆì— library ì¤€ë¹„(adptor ligation)ê¹Œì§€ í•˜ì§€ë§Œ, ì¢€ ë” sensitive(salt, handling ë“±)í•˜ê³  ë³µì¡í•¨. í•˜ì§€ë§Œ ì ì€ cell ìˆ˜ì™€ ë¹ ë¥´ê²Œ ì²˜ë¦¬ ê°€ëŠ¥. 
###### Cut-and-run 
Cleavage Under Targets & Release Using Nuclease

Chromatinì˜ Protein-DNA complex ë¶„ë¦¬. 
pAG-MNaseì™€ TF specific-ABë¡œ TFê°€ ë¶™ì€ ìœ„ì¹˜ë§Œ ì˜ë¼ë‚´ì„œ sesquencing
êµ³ì´ TF ì•„ë‹ˆë”ë¼ë„, histone mod(PTM) ë“±ë„ ê°€ëŠ¥!

CaCl2+ë¡œ MNase activation
##### DamID
to find LAD

##### CD Spectrography
CD : Circular Dimorphism
2nd structure í™•ì¸ ê°„ìœ¼
## Sequencing
### Sequencing Terms
##### Spike-in

- SIRVs
- ERCC

##### PTM
Post-Translation Modification

##### QTL
Quantitative trait loci
â†’ eQTL, sQTL, ie isâ€¦
##### eQTL
expression Quantitative trait loci
ìœ ì „ì ë°œí˜„ì— ì˜í–¥ì„ ì£¼ëŠ” ì„œì—´ ë³€ì´ê°€ ì¡´ì¬í•˜ëŠ” ê²Œë†ˆ ì˜ì—­
##### Sequins
 [Sequins](https://www.nature.com/articles/s41596-019-0175-1) : Sequencing spike-ins

##### T2T
=Telomere to Telomere


##### Dwell time
 Nanopore seqì—ì„œ pore í†µê³¼ ì‹œê°„ì´ ê¸¸ë©´ ê°™ì€ valueê°€ ì°íˆëŠ” ì‹œê°„ ë‹¤ë§Œ poreì— ìˆëŠ” sensing regionê³¼ motor/helicase ìœ„ì¹˜ëŠ” 12bpì •ë„ ì°¨ì´ ìˆìŒ

##### UMI
Unique Molecular Identifier 
scRNAseq ë“±ì—ì„œ ì—¬ëŸ¬ ì¢…ë¥˜ë¡œ ë ˆì´ë¸”ëœ markerë¡œ ì†ì‹¤ ì „ mRNA seq ì–‘ ë³´ê¸° ~~ìš°ë¯¸-ë§Œë©˜ë¯¸~~

##### Discordant read
<> Concordant(Proper) read
1. Read 1 and 2 have unexpedted mapping distance
	1. R1 ------â†’ Long unexp. dist â†-------R2
	2. ex) Genome deletion
2. Read 1 and 2 orientation are not matching
	1. R1------â†’ R2------â†’(FF)
	2. ex) Read on Duplicated genome boundary / reverse-duplicated
	3. R1â†------- ------â€“>R2
	4. ex) Read on duplicated genome boundary 
3. 

##### Chimeric read
~ Split reads, Supplementary alignment
í•˜ë‚˜ì˜ readê°€ ì„œë¡œ ë‹¤ë¥¸ ë‘ genome ì˜ì—­ì— mapping
One of chimeric read will be represented as linear alignment

##### Representative alignment
<> Supplementary alignment
Linear alignment ì¤‘ ëŒ€í‘œì„±ì„ ë³´ì´ëŠ” alignment
### Sequencing Methods

##### Illumina NGS
Sequencing by Synthesis(SBS) / Bridge PCR(ExAmp)

- í˜•ê´‘ dye label +  nucleotides
- ê¸¸ì´ ê¸¸ì–´ì§€ê±°ë‚˜, í˜•ê´‘ì„ ì–µì œí•˜ëŠ” ê²½ìš°ê°€ ì¡´ì¬

| 5'  | P5 Adaptor   | Index 1(I7) | Rd1 Primer site | (UMI) | Insert | (UMI) | Rd2 Primer site | Index 2(I5) | P7 Adaptor   | 3'  |
| --- | ------------ | ----------- | --------------- | ----- | ------ | ----- | --------------- | ----------- | ------------ | --- |
|     | Flow Cell ë¶€ì°© | Lib Index   | Amp Primer      |       |        |       | Amp Primer      | Lib Index   | Flow Cell ë¶€ì°© |     |
- P5/P7 ë”°ë¼ ë¶€ì°© í›„
- Index runì—ì„œ Index read
- Main PCRì€ Primer siteë¡œ Insert Amp
- 



##### Pacbio ONSO
Sequencing by Binding(SBB) / p;re-PCR

##### DNA Nanoball
= DNBseq (BGI/MGI) / Rolling circle amp
Sequencing by Ligation

##### SMRT
SMRT(Single-molecule Real-time sequencing) : PacBioì—ì„œ. í˜•ê´‘?

##### DRS
Direct RNA Sequencing
í˜„ì¬ ì“°ëŠ” Nanopore Sequencingì²˜ëŸ¼ PCR-free / Direct RNAë¥¼ ì½ëŠ” sequencing <->Long/Short read
Align í›„ quantification(ë¹„êµ ìœ„í•´ì„œ)ì—...

##### CCR

## Immunology
WIP ğŸ› ï¸

HPS(Hematopoietic Stem Cell) â†’ CLP(Common Lympohoid Progenitor) or CMP(Comon Myeloid Progenitor)
CLP â†’ T/B, Lympoids
CMP â†’ Dendrocyte, Granulocyte, Monocyteâ†’Macrophage, Platelets, RBCâ€¦
 are called Effector.

Primary Lympoid Tissue : Bone marrow and Thymus
Secondary Lympoid Tissue : Lymph nodes (50% are @ Intentines, Splean)

Innate Immunology : Avoidance(Skin, mucosa barriers) / Resistance(AMP-Antimicrobial proteins like C3, defensins & Innate Immune cells like Macrophage, NK, granulocytes), Tolerance(Adaptive immunity by B and T cells)

Inflammatory Inducer : LPS(Lipopoly saccaride), urate crystals
â†’
Sensor cells : Macrophage(Mâˆ…), neutrophils, dendritic cells(DC) : Use PRR(Pattern Recognition Receptors) <> PAMPs(Pathogen Associated Molecular Pattern)
â†’
Inflammatory mediators : Cytok,ine, Cytotoxicity â†’ Propagate and kill

Pathogen groups
- Virus, Bacteria, Fungi(Microorganisms) and Protozoa, Helminths(Parasites)
- Location : Extracellular(< AMP, Ab) / Intracellular(Cytoplasmic < NK, CTL / Vesicualr < Mâˆ…)
- Damage or Infection?
	- Direct damage : Exotoxin, Endoxotin, Direct
	- Indirect damage : Immune copmles, Anti-host Ab, Cell-mediated damage

### Innate Immunology



## Genes

### Base Modification

#### DNA Modification

- 5mC : 5-methylcytosine by DNMTs
- 5hmC : 5-hydroxymethylcytosine by TET1
- 5fC : 5-formylcytosine
- 5caC : 5-carboxylcytosine
#### RNA modification
- 6mA N6-Methyladenosine

- m6Am : N6,2'-O-dimethyladenosine

- 1mA : N1 methyladenosine

- 5mC : 5-methylcytosine

- hm5C : 5-hydroxymethylcytosine

- ac5C : N5 acetylcytidine

- m7G : N7-methylguanosine

-  Î¨ pseudouridine

  ?

- 4sU?
	-  TT-seq

- 6SG 6-thioguanosine

- 5eU 5-ethynyl uridine (5-EU)

- 

#### Nucleoside modification
  - 2' O-methyl (Nm)

### Variants
##### Allele 
- Chromosome ìƒì—ì„œ ê°™ì€ locusì— ì¡´ì¬í•˜ëŠ” 2ê°œ ì´ìƒì˜ geneì´ ìˆìŒ
	ë‹¹ì—°íˆ Humanì€ 2ê°œ(ì¼ë°˜ì ìœ¼ë¡ )
	ê·¸ê²ƒì˜ Variant.

##### Allele Ferquency
- íŠ¹ì • populationì—ì„œ, íŠ¹ì • locusì˜ geneì˜ ì—¬ëŸ¬ Allele(Genomic variants) ì˜ ìƒëŒ€ì ì¸ ë¹ˆë„

##### Allele Imbalance
- ë‘ allele ê°„ ë°œí˜„ì˜ ì°¨ì´

##### VAF
- Variant Allele Frequency
- ì´ê±´ Populationì´ ì•„ë‹ˆë¼, Tumor sample ê°™ì€ ë¹„êµ ì‹œì— íŠ¹ì • alleleì´ ë‚˜íƒ€ë‚˜ëŠ” variant read / ì „ì²´ read(reference + variant) .
- í˜¹ì€ N of allele / total occurance of all allele

##### BAF
- B-allele frequency
- allelic intensity ratio of two allele
- íŠ¹ì • alleleì„ A, ë‹¤ë¥¸ alleleì„ Bë¡œ ë‘ê³ 
	- AA : 0
	- AB : 0.5
	- BB : 1
	- AAB : 0.33

##### Copy Number


# Computer

##### FONC(First Order Necessary Conditions)
$fâ€™(x) = 0$
ê·¹ì (ì •ìƒì )ì´ê¸° ìœ„í•œ í•„ìš”ì¡°ê±´(aâ†b)

##### SONC(Second Order Necessary Conditions)
$fâ€™â€™(x) â‰¥ 0$
ê·¹ì†Œì´ê¸° ìœ„í•œ í•„ìš”ì¡°ê±´

##### SOSC(Second Order Sufficient Conditions)
$fâ€™â€™(x) > 0$
ê·¹ì†Œì´ê¸° ìœ„í•œ ì¶©ë¶„ì¡°ê±´(aâ†’b)

##### Amortized analysis

## Statistics

**==A . í•­ìƒ Relation(Corellation)ê³¼ Casual Inference(Casuality)ë¥¼ ì£¼ì˜í•  ê²ƒ.
B . ê³ ì „ì ì¸ Testë“¤ì˜ ê²½ìš° ê°€ì •í•˜ê³  ìˆëŠ” ë¶€ë¶„(ì •ê·œì„± ë“±)ì„ ë¯¸ë¦¬ í…ŒìŠ¤íŠ¸í•  ê²ƒ.
C . Rule of Thumb : CLT==**

Relationì´ ìˆë‹¤ê³  í•´ì„œ Casualtiyê°€ ì¡´ì¬í•œë‹¤ê³  ì¶”ì •í•  ìˆ˜ ì—†ìœ¼ë©°, ì´ë¯¸ ë°ì´í„° ë‚´ì˜ Endogenity ë“±ìœ¼ë¡œ ì¸í•œ ì˜í–¥ì´ ìˆìŒ.
##### Independent Variable
=Explanatory Variable
ë…ë¦½ë³€ìˆ˜/ì›ì¸ë³€ìˆ˜
##### Dependent Variable
=Predictor Variable
ì¢…ì†ë³€ìˆ˜/ì˜ˆì¸¡ë³€ìˆ˜

**Hyphothesis Test**
- Parametric Test(ëª¨ìˆ˜ ê²€ì •)
	- One sample
		- T-test
		- Z-test
	- Two samples
		- Indep.samples
			- Two-group T
			- Z
		- Paired samples
			- Paired T
- Nonparametric Test(ë¹„ëª¨ìˆ˜ ê²€ì •)
	- One sample
		- Chi-square
		- KS test
		- Binomial test
	- Two samples
		- Indep.samples
			- Chi-square
			- Mann-Whitney U
			- Median
			- KS
		- paired sample
			- Sign
			- Wilcoxon
			- McNemar
			- Chi-square
- Wald test?


| Notations              | Sample              | Population    |
| ---------------------- | ------------------- | ------------- |
| Datapoint              | $x_i$               | $x_i$         |
| Dataset size           | n                   | N             |
| Average                | $\bar{x}$           | $\mu$         |
| Std                    | $s$, $\hat{\sigma}$ | $\sigma$      |
| Variance               | $s^2$               | $\sigma^2$    |
| Regression coefficient | $\beta$             | $\hat{\beta}$ |
| Relation coefficient   | $\rho$              | $r$           |

```mermaid
flowchart TD
Â  Â  DV{Dependent Variable}
Â  Â  DV --> |Continous| Cont_ni{Independent Variable}
Â  Â  DV --> |Categorical| Clas_ni{Independent Variable}

Â  Â  Cont_ni -->|1| Cont_one{Independent Variable}
Â  Â  Cont_ni -->|2+| Cont_mul{Independent Variable}
Â  Â  Clas_ni --> |Categorical| Cat_n{Sample size}
Â  Â  Clas_ni --> Â Losgistic_regression[Logistic Regression]
Â  Â  
Â  Â  Cont_one --> |Categorical| Cont_one_cat{N. of groups}
Â  Â  Cont_one --> |Continious| SRA[SRA]
Â  Â  Cont_mul --> |Categorical| ANOVA[ANOVA]
Â  Â  Cont_mul --> |Mixed| ANCOVA[ANCOVA]
Â  Â  Cont_mul --> |Continious| MRA[MRA]
	Cat_n --> |small| Fisher_exact[Fisher's Exact test]
	Cat_n --> |large| Chi_square[Chi-Square Test]
	
Â  Â  Cont_one_cat --> |2| Cont_one_cat_two{Independency}
Â  Â  Cont_one_cat --> |3+| Cont_one_cat_mul{Normality}

Â  Â  Cont_one_cat_two --> |O| Cont_one_cat_two_ind{Normality}
Â  Â  Cont_one_cat_two --> |X| Cont_one_cat_two_nnd{Normality}
Â  Â  Cont_one_cat_mul --> |O| OW_ANOVA[One-Way ANOVA]
Â  Â  Cont_one_cat_mul --> |X| Kruskal_wallis[Kruskal-Wallis test]
Â  Â  Cont_one_cat_mul --> |X| Mood[Mood's Median]

Â  Â  Cont_one_cat_two_ind --> |O| Indep_T[Independent T-test]
Â  Â  Cont_one_cat_two_ind --> |X| Mann_U[Mann-Whitney U-test]
Â  Â  Cont_one_cat_two_nnd --> |O| Paired_t[Paired T-test]
Â  Â  Cont_one_cat_two_nnd --> |X| Wilcoxon[Wilcoxon signed-rank test]
```

 Average
	- Independent T-test
		- =Unpaired T test
		- If two groups fufills Normality and Independency, Homogenity of variance
		- H0 = â€œAverage of two group is sameâ€
			- ë‘ ê·¸ë£¹ì˜ í‰ê· ì€ ê°™ë‹¤
	- Mann-Whitney U Test
		- =Wilcoxon Rank sum test
		- If two groups are not fufills normality but independent
		- non-parametric test
		- H0 = â€œdistributions are sameâ€
			- ë‘ ê·¸ë£¹ì˜ ë¶„í¬ê°€ ë™ì¼í•˜ë‹¤(=ì¤‘ì•™ê°’ì´ ê°™ë‹¤)
	- Paired T-test
		- If two groups are Paired(Not independent) and fufills normality(normality of differences)
			- Paired : ê°™ì€ ëŒ€ìƒì˜ ì „-í›„ ë°ì´í„°(2ë²ˆ ì¸¡ì •í•œ ë°ì´í„° ê·¸ë£¹)
		- H0 = â€œ$\mu_d = 0$â€, Average of differences are 0
			- ë‘ ì¸¡ì •ê°’ ê·¸ë£¹ì˜ í‰ê·  ì°¨ê°€ ì—†ë‹¤
	- Wilcoxon signed-rank Test
		- If two groups are not independent and not normal
		- non-parametric test of paired-T test but *paired and ordered data*
		- H0 = â€œ$\mu_d = 0$â€, Average of differences are 0
			- ë‘ ì¸¡ì •ê°’ êµ¬ë¦…ê°„ ì°¨ì˜ ì¤‘ì•™ê°’ì´ 0ì´ë‹¤
			- Ordianal ë°ì´í„°ë„ ê°€ëŠ¥í•´ì„œ ìƒëŒ€ì ìœ¼ë¡œ ì´ìƒì¹˜ì— ëœ ë¯¼ê°í•¨
	- Friedman test
		- Extended Wilcoxon signed-rank test
		- If many groups are not independent, and non-parametric
		- H0 = â€œRank of each variables is same between groupsâ€
			- ê·¸ë£¹ ê°„, ë³€ìˆ˜ ìˆœìœ„ì˜ ì°¨ì´ê°€ ì—†ë‹¤
	- Kruskal-Wallis H test
		- Median version of Mann-whitney U test
		- Samples are independent but not normal
		- H0 = â€œMedian are same between all groupsâ€
			- ëª¨ë“  ê·¸ë£¹ì˜ ì¤‘ì•™/ì¤‘ìœ„ê°’ì€ ê°™ë‹¤
	- Moodâ€™s Median test
		- =Wilcoxon-Mann-Whitney test
		- Median version of Mann-whitney U test
			- More robust to outliers than Kruskal-wallis
		- Samples are paired, independent and ordinal but not normal.
		- H0 = â€œMedian of all groups are sameâ€
			- ëª¨ë“  ê·¸ë£¹ì˜ ì¤‘ì•™ê°’ì´ ê°™ë‹¤
	- One-way ANOVA
		- Multiple groups, Independent and normal dist, and same variance
			- Consider one Indep.variable
		- H0 = â€œThere is no difference between mean of groupsâ€
			- ê·¸ë£¹ë“¤ ê°„ í‰ê· ì˜ ì°¨ì´ê°€ ì—†ë‹¤
	- Two-way ANOVA
		- Multiple groups, Independent and normal dist, same variance
			- But consider TWO independent variables
		- H0_1 = â€œThere is no difference between groups about mean of factor Aâ€
			- ë…ë¦½ë³€ìˆ˜ Aì— ì˜í•œ ê·¸ë£¹ ê°„ í‰ê·  ì°¨ì´ê°€ ì—†ë‹¤
		- H0_2 = â€œThere is no difference between groups about mean of factor Bâ€
			- ë…ë¦½ë³€ìˆ˜ Bì— ì˜í•œ ê·¸ë£¹ ê°„ í‰ê·  ì°¨ì´ê°€ ì—†ë‹¤
		- H0_3 = â€œThere is no relationship between factor A and Bâ€
			- ë…ë¦½ë³€ìˆ˜ Aì™€ B ê°„ì˜ ê´€ê³„ê°€ ì—†ë‹¤
	- MANOVA
		- ë‹¤ë³€ëŸ‰(2+ dep variable) ì´ë‚˜ ê³µë³€ëŸ‰ì„ ê³ ë ¤í•œ ANOVA
		- Multiple categorical variables, but independent, Normal and homogenious variance
		- H0 : â€œâ€
- Post-hoc tests
	- Multiple groups(ANOVA)
	- Fisherâ€™s LSD
	- FDR 
	- Tukeyâ€™s range test
		- =Tukey-Kramer method, Tukey HSD test
		- <> Tukey mean-difference plot, Bland-altman plot
		- HSD(Honestly Significant Difference)
		- Also used in anomaly detection after ANOVA
	- Dunnettâ€™s test
	- Duncanâ€™s Multiple Range test
		- LSR(Least Significant Range)
		- more conservative than Tukeyâ€™s test
	- Bonferroni correction
	- Scheffe
	- Games Howell
		- If groups are not passes homogenity of variance test
		- = Welch test
		- 
- Variance tests
	- F test
		- Comparing the variance ratio between samples
			- F testë¼ëŠ” ì´ë¦„ì€ Varance í…ŒìŠ¤íŠ¸ë¥¼ í¬ê´„í•˜ëŠ” ëœ»ìœ¼ë¡œë„ ì“°ì„
			- Varianceì˜ ë¹„ê°€ F ë¶„í¬ë¥¼ ë”°ë¥´ëŠ”ì§€ì— ê´€í•´ ê²€ì •í•˜ë¯€ë¡œ, Normality(ì •ê·œì„±)ì— ì˜í–¥ì„ ë°›ìŒ. ì¼ë°˜ì ìœ¼ë¡œ, ì •ê·œì„±ì´ ìˆëŠ” ë…ë¦½ ê·¸ë£¹ì—ì„œ ì‚¬ìš©í•  ê²ƒ.
		- H0 = â€œVariance between two group is sameâ€
			- ë‘ ì§‘ë‹¨ì˜ ë¶„ì‚°ì´ ê°™ìŒ
	- Flinger-Kleeen test
		- Independent, multiple group but not normal(non-parametric)
			- H0 = â€œNo variance difference between groupsâ€
				- ê·¸ë£¹ ê°„ ë¶„ì‚° ì°¨ì´ê°€ ì—†ìŒ
	- Welchâ€™s test
	- Brown-forsythe test
- Homotenity of Varaince
	-  Levene test
		- Equality of variances between groups
		- H0 = â€œVariance between groups is sameâ€
		- Less senstive about normality than bartlettâ€™s test
	- Bartlettâ€™s test
		- Variances are homogenous(equal) between multiple grups
			- H0 = â€œPopulation variances are equalâ€
	- Hartleyâ€™s Fmax test
	- Oâ€™Brienâ€™s test
- Normality(Nonparametric? Parametric?)
	- Shapiro-Wilk
	- Kolmogorov-Smirnov
	- Anderson-Darling : Tail-Weighted KS test
	- Lilliefors : General, Modified p value of KS test 
- Goodness of fit
	- Cramer-von Mises
	- Likelihood-ratio test
- Equality of scale parameter
	- Ansari-Bradley
- Anomaly detection
	- Z-score
	- Corchranâ€™s C test : expected variance is significantly bigger than population variance?
	- Tukeyâ€™s range test(Post-hoc section)
	- Grubbsâ€™s test
		- Is dataset can be approximated by a normal dist?
		- H0 = â€œThere are no outliers in datasetâ€
- Categorical relationship
	- Chi-square test
	- Fisherâ€™s Exact test
- Structual change
	- Sup-Wald test
	- CUSUM test
	- Sup-LM test
	- Sup-LR test
	- Sup-MZ(Maasoumi-Zaman) test
- Others
	- Durbin-Watson test : Autocorrelation of Residuals
	- Davies test : on GLM, check slope, not breakpoint
	- Corchranâ€™s Q test : propertion of success across groups
	- Wald test : Estimation parameter test
	- Chow test : Comparing two linear models
	- SPRT(Sequential Probability Ratio test) : Sequential hypothesis test
	- Vuongâ€™s closeness test
	- Dickey-Fuller test
- Cointegration test
	- Engle and Granger test
	- Johansen test
	- 

##### Observation

| Observation(result) \ Truth(real) | TRUE(True set) | FALSE(False set) |
| --------------------------------- | -------------- | ---------------- |
| TRUE(Positive)                    | TP             | FP               |
| FALSE(Negative)                   | TN             | FN               |

##### Type I error
= False Positive = $\alpha$ error

##### Type II error
= False Negative = $\beta$ error

##### FDR
False Discovery Rate
$FDR = FP/TP$

##### CLT
Central Limit Theorem

##### Skewness
ì™œë„

##### Kurtosis
ì²¨ë„

##### IID
Independent and identically distributed (random  variables)
ë…ë¦½ì ì´ë©´ì„œ, ê°™ì€ í™•ë¥  ë¶„í¬ë¡œë¶€í„° ìœ ë˜í•œ random variableì´ë¼ëŠ” ëœ»

#### Bayesian Statistics
- Frequentist : ì´ëŸ° ë°ì´í„°ë¡œë¶€í„° ì–»ëŠ” distributionì˜ í™•ë¥ ë¶„í¬ëŠ”?
- Bayes : íŠ¹ì • distributionì—ì„œ ì´ëŸ° ë°ì´í„°ê°€ ë‚˜ì˜¬ í™•ë¥ ì€?



##### Bayes Rule
$P(A|B) = \cfrac{P(B|A)P(A)}{P(B)}$
ë³´í†µ modelì—ì„œëŠ” Aê°€ parameter $\theta$, Bê°€ ë°ì´í„° $D$ë¡œ ì“°ì„
í˜¹ì€, $\text{Posterior} = \cfrac{\text{likelihood} \times \text{prior}}{\text{evidence}}$ ë¡œë„ ë§í•¨
- lilkelihoodì— ëŒ€í•´, $P(D|\theta)$ê°€ ë¹„ ì§ê´€ì ?ì´ì–´ì„œ $L(\theta|D)$ë¡œë„ ì“°ëŠ” ê²½ìš°ê°€ ìˆìŒ
- Posteriorì—ì„œ Priorë¥¼ ì—…ë°ì´íŠ¸(ê´€ì¸¡ ê²°ê³¼ë¡œ, ë°°ê²½ ë¶„í¬ë¥¼ ê°±ì‹ )í•œë‹¤ë©´ Emperical Bayes
- Priorë¡œë¶€í„° Posteriorë¥¼ ì¶”ì •(ì•Œë ¤ì§„ ë¶„í¬ë¡œë¶€í„° ë°ì´í„°ë¥¼ ì¶”ì¶œ)í•œë‹¤ë©´ Likelihood ê¸°ë°˜ Inference


##### MLE
Maximum Likelihood Estimation
$\underset{\theta}{max}(L(\theta|x))$ í•˜ëŠ” $\theta$ëŠ”?

##### MAP
Maximum 
$\underset{\theta}{max}(P(\theta|x)) = \underset{\theta}{max}(P(X|\theta)P(\theta))$ 
Priorë¡œë¶€í„°

### Probabilty Distribution

##### Normal distribution

##### Z distribution
= Standarized Normal Distribution

##### T distribution
= Studentrized Normal Distribution

##### F distribution

##### Chi-Square distribution
$\chi^2$ distribution, 
##### Binomial distribution


### Correlation analysis
= Dependence analysis

##### Pearson Correlation

##### Spearman Correlation

##### Autocorrelation



### Parametric test
= ëª¨ìˆ˜ ê²€ì •

### Nonparametric test
= ë¹„ëª¨ìˆ˜ ê²€ì •.

### Significant Test

##### Z-Test

##### T-Test

##### Chi-Square Test

##### F-Test

##### Mann-Whitney U test
= Wilcoxon ranked sum

### Normality Test

##### Komolog-Smirnov test
= KS test

##### Shapiro-Wilk test

### Variance analysis

##### MANOVA
##### One-Way ANOVA

##### Two-way ANOVA

##### Repeated measures ANOVA

##### ANCOVA

##### Bonferroni correction

##### Benjamini-Hochberg Procedure


##### RM

##### Kruskal-Wallis
### Homoscedasticity test
##### Levene test
##### Barlett test

### Regression

##### VIF
Variance Inflation Factor = $\cfrac{1}{1-r_i}$
Multicolinearity(ë‹¤ì¤‘ê³µì„ ì„±)ì„ ë³´ê¸° ìœ„í•œ ì²™ë„

##### Durbin-Watson test
Linear regression modelì˜ Residual(ì˜¤ì°¨í•­)ì˜ Autocorrelation ì„ ë³´ëŠ” test
DW = 2 : No autocorrelation
DW < 2 : Positive autocorrelation
DW > 2 : Negative autocorrelation


## Web

##### CORS
Cross-Origin-resource-sharing
https://inpa.tistory.com/entry/WEB-%F0%9F%93%9A-CORS-%F0%9F%92%AF-%EC%A0%95%EB%A6%AC-%ED%95%B4%EA%B2%B0-%EB%B0%A9%EB%B2%95-%F0%9F%91%8F

##### Ajax
Asyncrnous Javascript transfer

## Signal processing

##### Nyquist Frequency
sampling speedì— ì˜í•´ ê²°ì •ë˜ëŠ” ì¸¡ì • ê°€ëŠ¥í•œ frequency ì˜ì—­ì˜ í•œë„

#### Filters

##### Savitzky-Golay filter

##### Butterworth filter

##### Kalman Filter

###### EKF

###### ESKF

###### IEKF

### Image processing

##### Adaptive threshold
##### Otsu binarization


##### Gaussian Pyramids

##### Laplacian Pyramids
##### CLAHE
Contrast Limited Adaptive Histogram Equalization
Kind of histogram equalization method.

##### Salt and Pepper

##### GLCM
= Gray-Level Cross Matrix


#### Morphologic Transformation
##### Erosion
##### Dilation
##### Opening
##### Closing

#### Gradient Filters

##### Sobel Filter

##### Scharr Filter

##### Laplacian Filter



#### Hough Transforms


## Reinforcement Learning

## Machine Learning
Main Tasks
- Supervised learning
	- Classification
	- Regression
- Clustering
- Dimension Reduction
- Anomaly detection
- Artificial Neural Network
- Reinforcement Learning
- 

ë‚˜ì¤‘ì— Supervised

ëŒ€ë¶€ë¶„ì˜ ìµœì í™” ê¸°ë²•ì—ëŠ” [[Terms#IID]] ì¶”ì •ì´ ë“¤ì–´ê°
ì¼ë°˜ì ìœ¼ë¡œ
- ê´€ì¸¡ì¹˜, ê´€ì¸¡ ë°ì´í„° $\mathcal{X} = set(X_i)$
- ì ì¬ë³€ìˆ˜, latent, missing values $\mathcal{Z}$
- ìµœì í™”í•˜ê³ ì í•˜ëŠ” íŒŒë¼ë¯¸í„° $\theta$
- ìš°ë„ í•¨ìˆ˜, Likelihood function $L(\theta;\mathcal{X},\mathcal{Z}) = p(\mathcal{X}, \mathcal{Z}|\theta)$

##### Curse of dimensionality

##### Bias-Variance tradeoff

##### MLE
Maximum Likelihood Estimation

##### EM
Expectation maximization algorithm
Local maximum likelihoodë¥¼ ì°¾ëŠ” ë°©ë²•ìœ¼ë¡œ ì‚¬ìš©ë¨
ë³´í†µ ëª¨ë¥´ëŠ” latent variable $\theta$ë¥¼ ì•Œê³  ìˆëŠ” ê´€ì¸¡ì¹˜ $Y$ë¡œ ì—…ë°ì´íŠ¸ í•  ë•Œ,
- ë¨¼ì € ì„ì˜ë¡œ $\theta$ë¥¼ ì„¤ì •í•œ ë’¤
- $\theta$ ë¡œë¶€í„° $Y$ê°€ ë‚˜ì˜¬ í™•ë¥ (Responsibility)ë¥¼ ê³„ì‚°(Expectation)
- ì–»ì–´ë‚¸ responsibilityë¡œë¶€í„°, $\theta$ë¥¼ ì—…ë°ì´íŠ¸.(Maximization)

##### SGD
Stochastic Gradient Descent
##### Boosting

##### Casuality
ì¸ê³¼ ë¬¸ì œ. MLì€ ëª»í•˜ì§€â€¦

##### Ensemble

##### Voting

##### RI
Rand Index

##### ZSL
=Zero-shot Learning

##### KL Divergence
=Kullback-Leibler Divergence (Relative entropy, I-divergence)
$D_{KL}(P||Q)$ 

ë‘ í™•ë¥ ë¶„í¬ ê°„ ì°¨ì´ë¥¼ ë¹„êµí•´ entropy ê³„ì‚°, ê°™ì„ ë•Œ 0 ë‹¤ë¥´ë‹¤ë©´ 
[[Terms#Likelihood]]ì™€ ë°˜ëŒ€ë¡œ ì´ë™
##### Jaccard staticita
=Jaccard Simillarity Index
$J(A,B) = \cfrac{A \cap B}{A\cup B}$

##### Likelihood
ìš°ë„, ê°€ëŠ¥ë„
Sampleì´ íŠ¹ì •í•œ í™•ë¥ ë¶„í¬(í˜¹ì€ model)ì—ì„œ ì™”ì„ í™•ë¥ 
ProbablityëŠ” í™•ë¥ ë¶„í¬ì—ì„œ íŠ¹ì • sampleì´ ë½‘í í™•ë¥ ë¡œ ë‹¤ë¥´ë‹¤!

##### L1 Norm
Manhattan distance, Taxicab geometry
$d_1(a,b) == ||a-b||_1 = \Sigma |a_i - b_i|$
ë‘ ë²¡í„° ì°¨ì˜ ì ˆëŒ€ê°’ì˜ í•©
##### L2 Norm
Euclidian distance
$d_2(a,b) = \Sigma\sqrt{(a_i-b_i)^2}$
##### L1 Loss
LAD,LAE, LAV, LAR
$L_1(x) = \Sigma|y_i-f(x_i)|$ 
$f(x)$ëŠ” $\hat(y)$ë¼ê³  ìƒê°í•´ë„ ë¨ (y_pred = f(x))
L2ì— ë¹„í•´ outlierì— ë” ê°•í•˜ë‚˜, 0ì—ì„œ ë¯¸ë¶„ ë¶ˆê°€
ë‹¤ë§Œ ì—¬ëŸ¬ solutionì„ ê°€ì§ˆ ìˆ˜ ìˆìŒ
##### L2 Loss
LSE(Least Square Error)
$L_2(x) = \Sigma(y_i-f(x_i))^2$
í•˜ë‚˜ì˜ solutionë§Œì„ ê°€ì ¸ stableí•˜ë‚˜, outlierì— ëŒ€í•´ ë” ë¶ˆì•ˆì •í•¨
##### Regularization
ì •ê·œí™”. 
Loss functionì— weightì„ ë”í•´ ë” robustí•œ ëª¨ë¸ì„ ìœ„í•´ ë„£ëŠ” ì¶”ê°€ í•­
ì£¼ë¡œ Overfitting ë°©ì§€ì— ì‚¬ìš©
##### L1 regularization
$Cost = \cfrac{1}{n}\Sigma(L_1(y_i, x_i)+\cfrac{\lambda}{2}|w|)$
ê²½ë¡œì™€ ìƒê´€ ì—†ì´ ì¤„ì´ëŠ” ë°©í–¥ìœ¼ë¡œ, featureë¥¼ ì£½ì´ëŠ” ê²½í–¥ì„±ì´ ìˆìŒ = ë¶ˆí•„ìš”í•œ featureë¥¼ ì§€ìš°ëŠ” íš¨ê³¼
â†’ Lasso model(Linear regression)
##### L2 regularization
$Cost = \cfrac{1}{n}\Sigma(L_2(y_i, x_i) + \cfrac{\lambda}{2}|w|^2)$
L1ê³¼ëŠ” ë‹¬ë¦¬ feature weightì´ í•­ìƒ >0. 
â†’ Ridge model(Linear regression)

##### MAE
Mean absolute Error
$MAE = \cfrac{1}{n}\Sigma|y_i-f(x_i)|$
ì´ìƒì¹˜ì— ë§¤ìš° ë‘”ê°í•¨
ëª¨ë“  ì˜¤ì°¨ì— ê°™ì€ ê°€ì¤‘ì¹˜ê°€ ë¶€ì—¬ë˜ë©°, 0ì—ì„œ ë¯¸ë¶„ ë¶ˆê°€ëŠ¥í•¨.
##### MSE
Mean square error
$MSE = \cfrac{1}{n}\Sigma(y_i - f(x_i))^2$
squareê°€ ìˆì–´ì„œ ì´ìƒì¹˜ì— ë¯¼ê°í•˜ê³  í¬ê²Œ ë°˜ì˜í•¨. íŠ¹íˆ, 0-1 ì‚¬ì´ì—” ë‘”ê°í•˜ê³  1 ì´ìƒì¼ë•Œ ë¯¼ê°í•¨
í•­ìƒ ë¯¸ë¶„ ê°€ëŠ¥í•¨
##### RMSE
Root Mean square error
$RMSE = \sqrt{\cfrac{1}{n}\Sigma(y_i-f(x_i))^2}$
MSE ë³´ë‹¤ëŠ” ëœ ë¯¼ê°í•´ ì´ìƒì¹˜ë¥¼ ë‹¤ë£° ë•Œ ë‚«ì§€ë§Œ, 0ì—ì„œ ë¯¸ë¶„ ë¶ˆê°€ëŠ¥í•¨

##### RMLSE

##### MAPE

##### OLS
Ordinary Least Square

##### Linear regression
Simple linear regression : $\hat y = Wx + b$
Multiple linear regression : $\hat y = \Sigma W_ix_i + b$
##### R square
$R^2$. linear regressionì—ì„œ ê²°ê³¼ì™€ ì›ë³¸ ì‚¬ì´ì˜ correlation
##### Ridge regression

##### Lasso regression

##### Elastic Net

##### Hessian-Eigenmap

##### Elbow method

##### Silhouette score
Clustering í‰ê°€ì— ì‚¬ìš©

##### Adjusted P value

##### Bonferroni-Hochberg Procedure

##### SOM algorithm
= Kohonen map

###### MiniSOM

##### GMM
Gaussian Mixture Model
ì£¼ì–´ì§„ ë°ì´í„° $\mathcal{X} = \{X_1, X_2... X_N\}$ê°€, $K$ê°œì˜ Gaussian distributionìœ¼ë¡œ ê·¼ì‚¬í•˜ëŠ” ê³¼ì •
ì¦‰, $p(X) = \Sigma^K_{k=1}{\pi_k\mathcal{N}(X;\mu_k;\Sigma_k)}$ ë¥¼ ìµœëŒ€í™”í•˜ëŠ” 3$K$ê°œì˜ íŒŒë¼ë¯¸í„° $\theta = \{\pi_k, \mu_k, \Sigma_k\}$ë¥¼ ì°¾ëŠ” ê³¼ì •
- 
- Mixing coefficient $\pi_k \in [0, 1]$, $\Sigma\pi_k = 1$

https://untitledtblog.tistory.com/133

### Data Integration

##### CCA
Canonical Correlation Analysis

##### MNN
Mutual Nearest Neighbor
ì„œë¡œ ê°€ê¹Œìš´ neighbor ìŒì„ ì°¾ì€ ë’¤, ê°™ì€ groupìœ¼ë¡œ ë³´ê³  projection ê³„ì‚° ë° ì´ë™
### Basics

##### Arithmetic mean
$\cfrac{a+b}{2} = \cfrac{\Sigma x_i}{n}$

##### Geometric mean
$\sqrt{a\cdot b}$

##### Harmonic mean
$\cfrac{2ab}{a+b}=\cfrac{1}{Mean(\cfrac{1}{x_i})}$

##### Confusion matrix
~~ë§ ê·¸ëŒ€ë¡œ í•­ìƒ ë´ì•¼ í•˜ëŠ” matrix~~

| Observation(result) \ Truth(real) | TRUE(True set) | FALSE(False set) |             |
| --------------------------------- | -------------- | ---------------- | ----------- |
| TRUE(Positive)                    | TP             | FP               | Sensitivity |
| FALSE(Negative)                   | TN             | FN               | Specificity |
|                                   | Precision      | NPV              | Accuracy    |
##### Bias
##### Variance

##### Sensitivity
=Recall
$=\cfrac{TP}{TP+FN} = \cfrac{TP}{Truth}$
ì‹¤ì œ Truth ì¤‘ ì–¼ë§ˆë‚˜ ì˜³ê²Œ ì˜ˆì¸¡í•˜ëŠ”ê°€?
##### Precision
=PPV(Positive predective value)
$=\cfrac{TP}{TP+FP} = \cfrac{TP}{Positive}$
Trueë¼ê³  ì˜ˆì¸¡í•œ ê²ƒ ì¤‘ ì‹¤ì œ Truthì€ ì–¼ë§ˆë‚˜ ìˆëŠ”ê°€?
(Sensitivtyì™€ Precisionì€ tradeoff)

##### F1 score
=Dice score
$F_1=\cfrac{2}{\cfrac{1}{recall}\times\cfrac{1}{precision}}=2\times\cfrac{precision\times recall}{precision + recall} = \cfrac{TP}{TP+\cfrac{1}{2}(FP+FN)}$
$Dice(A,B) = \cfrac{2\times|A\cap B|}{|A|+|B|}$

##### Specificity
$=\cfrac{TN}{FP+TN}=\cfrac{TN}{Error}$
ì–¼ë§ˆë‚˜ Falseë¥¼ ì˜³ê²Œ ì˜ˆì¸¡í–ˆëŠ”ê°€?

##### NPV
Negative predicted value
$=\cfrac{TN}{TN+FN}=\cfrac{TN}{Negative}$
Negative ì˜ˆì¸¡ ì¤‘ ì§„ì§œ FalseëŠ” ì–¼ë§ˆë‚˜ ìˆì—ˆëŠ”ê°€?
##### Accuracy
$=\cfrac{TP+TN}{TP+FP+FN+TN}=\cfrac{Truth}{\Sigma}$

##### ROC curve
Threshold ë³€í™”ì— ë”°ë¥¸ 1-specificity - sensitivity plot
###### ROC-AUC

###### PR-AUC
Threshold ë³€í™”ì— ë”°ë¥¸ sensitivity - precision plot
### Batch correction

##### MNN

##### ComBat

### Dimension Reduction

##### PCA
Principle Component Analysis

##### ICA
Independent Component Analysis

##### LDA
Latent Dirichlet Allocation

##### tSNE
t-distributed Stochastic Neighbor Embedding

##### UMAP
Uniform Manifold Approximation and Projection
Dataë¥¼ Simplexë¡œ ìƒê°í•˜ê³ , kê°œì˜ ì´ì›ƒ(k-simplexê°€ ë˜ëŠ” data)ë¥¼ max, ìµœê·¼ì ‘ ì´ì›ƒì„ minìœ¼ë¡œ í•˜ëŠ” fuzzy-sphere(ê±°ë¦¬ í•¨ìˆ˜)ë¥¼ ê°€ì§€ëŠ” edge-local neighbor ë¶„í¬(ê±°ë¦¬<ì´ì›ƒ) ì™€ ìœ ì‚¬í•œ ì €ì°¨ì› ë¶„í¬ë¥¼ ì°¾ëŠ” ê³¼ì •

Check nerve theorem[https://en.wikipedia.org/wiki/Nerve_complex] for more info!
##### PHATE
Potential of Heat-diffusion for Affinity-based Trajectory Embedding
##### LLE
Locally Linear Embedding
##### ISOMAP

### Clustering

##### Modularity
ë„¤íŠ¸ì›Œí¬ ë‚´ ì»¤ë®¤ë‹ˆí‹°(í´ëŸ¬ìŠ¤í„°) ê°„ ë§í¬ê°€ ë¬´ì‘ìœ„ì ì¸ ì—°ê²°ë³´ë‹¤ ì–¼ë§ˆë‚˜ ë§ì€ì§€ ë¹„êµ
$Q = \cfrac{1}{2M} \sum_{i, j}^{N} (a_{ij} - <t_{ij}) \delta [C(i), C(j)] = Tr(E) - ||E^2||$ 
- Q : Modularity
- M : Total link N
- N : Total Node N
- a_ij : link between i-j (0 or 1)
- t_ij : ê° ë…¸ë“œì˜ ë§í¬ ìˆ˜ë¥¼ ìœ ì§€í•œ ì±„ë¡œ, ë¬´ì‘ìœ„ë¡œ ì¬ì—°ê²°í–ˆì„ ë•Œ i-jê°„ ë§í¬ê°€ ìˆì„ ë•Œ(0 or 1)
- <t_ij> : t_ijì˜ expectation
- C(i) : ë…¸ë“œ iì˜ commiunity(cluster)
- $\delta[c(i),C(j)]$ : iì™€ jì˜ communityê°€ ê°™ì„ ë•Œ 1(0 or 1)
- E : ì»¤ë®¤ë‹ˆí‹° ê°„ì˜ ì—°ê²° ê°œìˆ˜ì˜ í–‰ë ¬
- Tr(E) : ëŒ€ê°í–‰ë ¬ì˜ í•©

https://mons1220.tistory.com/271?category=804486

##### Centrality
ì¤‘ì‹¬ì„±
https://mons1220.tistory.com/253?category=804486
###### Degree centrality
###### Eigenvector centrality
##### Katz centrality

##### PageRank centrality

###### Subgraph centrality
##### Lovain clustering

##### Girvan-Newman Algorithm
Betweenness centralityê°€ ë†’ì€ ì—°ê²°ë¶€í„° ì œê±°í•˜ë©´ì„œ ì»¤ë®¤ë‹ˆí‹°ë¥¼ ë§Œë“œëŠ” ì•Œê³ ë¦¬ì¦˜.
###### Betweeness centrality
ë„¤íŠ¸ì›Œí¬ ë‚´ ë…¸ë“œ iì™€ jë¥¼ ì—°ê²°í•˜ëŠ” ìµœë‹¨ ê±°ë¦¬ìƒ íŠ¹ì • kê°€ ì–¼ë§ˆë‚˜ ë§ì´ ë“±ì¥í•˜ëŠ”ì§€.

##### Leiden clustering

##### DBSCAN

##### HDBSCAN
### NLP
Natural Language Processing
##### Entropy

##### TF-IDF
Term frequency - Inverse Document Frequency

##### LDA
Latent Dirichlet Allocation

### Methods

##### RANSAC

##### Cross Validation

##### Grid Search

##### Decision Tree

##### Random Forest

##### HMM
Hidden Markov Model

Markov chain(íŠ¹ì • state transistionì— ì˜í–¥ì„ ì£¼ëŠ”ê±´ ì´ì „ stateë°–ì— ì—†ìŒ) ì— ê¸°ë°˜
ë³´í†µ ëª¨ë¸ $\lambda$ ëŠ” íŒŒë¼ë¯¸í„°ë¡œ $(A, B, \pi)$ë¥¼ ê°€ì§
- $A$: Transistion probability matrix
- $B$: Output probability(ê° stateì—ì„œ output observation $O$ë¥¼ ì¶œë ¥í•  í™•ë¥ )
- $\pi$: Initial state probability

ì´ê±¸ë¡œ ë³´í†µ ì„¸ê°€ì§€ ì‘ì—…ì— ì‚¬ìš©
- Known Observation $O=(o_1â€¦)$  and model $\lambda$ â†’ $P(O|\lambda)$? Forward-backward algorithm ì‚¬ìš©
- Known Observation $O=(o_1â€¦)$ and model $\lambda$ â†’ Latent state $Q=(q_1â€¦)$? Viterbi algorithm ì‚¬ìš©
- Known set of Observations $O_i$ â†’ Model $\lambda$? : EM ì‚¬ìš©


##### MEM
Maximum Entropy Models

##### MEMM
Maximum Entropy Markov Models
CRFì— ë¹„í•´ ë” localì—ì„œ ì •ê·œí™”
ë‘˜ ë‹¤ discriminate, Maximize Entrop;y ê¸°ë°˜
##### CRF
Conditional Random Field
https://ratsgo.github.io/machine%20learning/2017/11/10/CRF/


### XAI

Captum ì°¸ì¡°

##### ICE
##### SHAP
SHapley Additive exPlanation
##### LIME
=Local Interpretable Model-agnostic Explanations

##### EBM
Explanable Boosting Machine
ìœ„ LIME + SHAP ì €ìê°€ ë§Œë“  Tree ê¸°ë°˜ Interpretable ML.(Glassbox Model)

##### Salincy Map

##### Integrated Gradients(IG)

##### Attention Matrix
https://arxiv.org/abs/1902.10186

### Deep Learning

#### Basics

##### Backpropagation

##### Vanishing Gradient



##### CNN
###### Stride

###### SE layer
Squeeze-and-Exciation layer

###### Depthwise-Pointwise convolution
Mobilenetì—ì„œ ì‚¬ìš©ëœ CNN ê²½ëŸ‰í™” ê¸°ë²•

###### Dilated convolution

##### RNN
Recurrent Neural Network

###### LSTM

###### GRU

##### SNN
Spiking Neural Network

##### Quantization

##### Transformer

###### KV Cache
Key-Value ìŒì€ Transformer ëª¨ë¸ì—ì„œ ë³€í™”ê°€ ì ê¸°ì—, ë³´í†µ ì´ê±¸ ìºì‹±í•´ì„œ ì„±ëŠ¥ì„ ëŠ˜ë¦¼
Computation-Boundì—ì„œ Memory-boundë¡œ ë„˜ì–´ê°„ ì£¼ ì´ìœ 

##### Diffusion
Noise + (Feature) ìŒ ì…ë ¥ìœ¼ë¡œ í›ˆë ¨ëœ ëª¨ë¸ì„ í†µí•´, ì„ì˜ì˜ ì¶œë ¥ì„ ìƒì„±í•˜ëŠ” ëª¨ë¸

##### RAG
Retrieval-Augmented Generation
LLMì˜ ë‹µë³€ ìƒì„± ì‹œ, ì ì ˆí•œ dbë¡œë¶€í„° ì •ë³´ë¥¼ ê²€ìƒ‰í•´ ì‚¬ìš©. 
ë³´í†µ db ì „ì²´ë³´ë‹¤ëŠ”, ë¦¬ë­ì»¤ë¥¼ í†µí•´ ì •ë ¬/í•„í„°ë§ëœ ê²°ê³¼ ì¼ë¶€ë¥¼ ê°€ì ¸ì™€ LLMì´ ì‚¬ìš©í•¨
#### Anomaly detection

##### Isolation Forest


#### Classification
##### SVM
Support Vector Machine
# Bioinfo

##### CPM
Counts per million mapped reads
$\text{CPM}=\cfrac{\text{read count}}{\text{total read counts}}*10^6$
PCR reads $\propto$ transcript length

##### RPKM
Reads per kilobase of tarnscript per million mapped reads
$\cfrac{\text{read count}}{\text{gene length} * \text{total read counts}} * 10^9$


##### FPKM
Fragments per kilobase of transcript per million mapped reads
RPKM ì—ì„œ Read ê°€ ì•„ë‹ˆë¼ complementary read=fragment ì˜ count ì‚¬ìš©

##### TPM
Transcripts per Million
$\cfrac{\text{read count}}{\text{gene length} * \Sigma(\frac{\text{read count}}{\text{gene length}})}*10^9$
total read countê°€ ì•„ë‹ˆë¼ TPM í•©ì´ 1ë¡œ ê³ ì •ë¨

##### CIGAR string
https://samtools.github.io/hts-specs/SAMv1.pdf
check [[SAM(BAM)]] format

| Op  | BAM | Desc                                      |
| --- | --- | ----------------------------------------- |
| M   | 0   | Alignment match                           |
| I   | 1   | Insertion to reference                    |
| D   | 2   | Deletion from reference                   |
| N   | 3   | Skipped region from reference             |
| S   | 4   | Soft clipping(clipped present in SEQ)     |
| H   | 5   | Hard clipping(clipped NOT present in SEQ) |
| P   | 6   | Padding(silent deletion from padded seq)  |
| =   | 7   | Sequence match                            |
| X   | 8   | Sequence mismatch                         |

##### IUPAC code
IPUAC Ambiguity code

| Symbol | Bases            | Origin of designation              |
| :----- | :--------------- | :--------------------------------- |
| G      | G                | Guanine                            |
| A      | A                | Adenine                            |
| T      | T                | Thymine                            |
| C      | C                | Cytosine                           |
| R      | G or A           | puRine                             |
| Y      | T or C           | pYrimidine                         |
| M      | A or C           | aMino                              |
| K      | G or T           | Keto                               |
| S      | G or C           | Strong interaction (3 H bonds)     |
| W      | A or T           | Weak interaction (2 H bonds)       |
| H      | A or C or T      | not-G, H follows G in the alphabet |
| B      | G or T or C      | not-A, B follows A                 |
| V      | G or C or A      | not-T (not-U), V follows U         |
| D      | G or A or T      | not-C, D follows C                 |
| N      | G or A or T or C | aNy                                |

##### PWM
Position Weight Matrix.
(ì „ê¸°ì „ìê³µí•™ì—ì„œëŠ” Pulse Width Modulation)

##### PSSM
Position specific scoring matrix

##### BWT
Burrows-Wheeler Transform
ìµœê·¼ Alignerì˜ ê·¼ë³¸ì´ ë˜ëŠ” indexing ê´€ë ¨ algorithm.

ì„œì—´ì„ query í•˜ê³ ì í•˜ëŠ” ì„œì—´(key ì•„ë‹˜)ì— ëŒ€í•´, ê°€ì¥ ì• í˜¹ì€ ë’¤ì— placeholder(ë³´í†µ `$`)ë¥¼ ë¶™ì¸ queryì˜ ëª¨ë“  ìˆœì„œìŒ(cyclic rotation)ì„ ê°€ì§€ëŠ” NxN matrixë¥¼ ë§Œë“¤ê³  ì´ë¥¼ indexì™€ í•¨ê»˜ sortingí•˜ë©´, sorted matrixê°€ ë‚˜ì˜´
ì´ sorted matrixì—ì„œ LF mappingì„ ìˆ˜í–‰í•˜ë©´ íŠ¹ì • keyë¥¼ ê°€ì§€ëŠ” subsequenceë¥¼ ê²€ìƒ‰í•  ìˆ˜ ìˆìŒ
##### Fold change
= FC
Expression levelê³¼ ëŒ€ìƒ mRNA countì˜ relationì´ ìˆë‹¤ê³  ìƒê°í•˜ê³  ì‚¬ìš©í•˜ëŠ” ê°’. Control ê³¼ expermientalê°„ expression levelì˜ ë³€í™”.
ë³´í†µ ê°’ì´ ì§€ìˆ˜ì ìœ¼ë¡œ ë‚˜íƒ€ë‚˜ì„œ, logì„ ì”Œìš´ log fold changeë¥¼ ë§ì´ ë´„.
ê´€ë ¨í•´ì„œ [[Terms#Volcano plot]]ì´ë‚˜ [[Terms#MA plot]] í™•ì¸
##### GWAS
Genome-wide Association Study
##### MPRA
Massively parallel report assay

##### GO
Gene Ontology

##### Phred scale
$\text{Phred} = -10 \times log_{10}(\text{Prob})$

- ìœ ë… BAM íŒŒì¼ ìƒì—ì„œ ìˆ˜ì¹˜ê°€ bias ë˜ì–´ ìˆëŠ”ë°, ASCII ë•Œë¬¸. +33 ì´ë‹¤.
	- ì˜›ë‚  Illumina callerì—ì„  64ì¸ê°€ ê·¸ëŸ°ë°, ë³¼ì¼ì´ ê±°ì´ ì—†ìŒ.
### Tools
Check tools folder

#### Basecaller

#### QC

#### Aligner

#### DB


### Alignment

##### Hamming Distance

##### Levenshtein Distance

##### Jaro-winkler Distance
aka. jaro-winkler similarity
ë‘ Stringê°„ similiartyë¥¼ êµ¬í•˜ëŠ”ë°, transposition(swapping)ì„ ê³ ë ¤í•˜ë©´ì„œ.


##### Global Alignment

##### Local Alignment

##### MSA
Multiple Sequence Alignment

### Visualization

##### COSMIC signature
COSMIC DBì—ì„œ ì œê³µí•˜ëŠ” ì—¬ëŸ¬ íŠ¹ì§•ì ì¸ mutation signature.(ì£¼ë¡œ tumor)
Base modificationì˜ ratio ë“±ìœ¼ë¡œ ì œê³µë¨.
NMF(Nonzero Matrix Factorization) + min(KL Divergence)ë¡œ ê³„ì‚°ë¨(Sigprofiler)

##### Sequence logo

##### Circos plot

##### Volcano plot
log fold changeì™€ p valueë¡œ ê·¸ë¦° plot

##### MA plot
log mean averageì™€ log ratio(log A -log B)ë¡œ ê·¸ë¦° plot
ì£¼ë¡œ scRNAseq ë¶„ì„ ì‹œ ë‘ ì§‘ë‹¨ ê°„ ë¹„êµì— ì‚¬ìš©
ë³´í†µ housekeeping geneì€ xì¶•ì´ ë†’ê³  yê°€ ë‚®ê²Œ, ì§‘ë‹¨ ê°„ ì°¨ì´ê°€ ë‚˜ëŠ” geneì€ yì¶• ì°¨ì´ê°€ í¼

##### Elbow plot
PCAì—ì„œ T-valueë¡œ ê·¸ë ¤ì„œ ì–´ëŠ axisê¹Œì§€ ìœ íš¨í•  ì§€ ë³´ê±°ë‚˜

##### Manhattan plot
GWASì—ì„œ SNPë¥¼ ê·¸ë¦¬ëŠ” scatterplot.

##### Kaplan-Meier curve
= Produce Limit Method / Kaplan-Meier Estimation
ìƒì¡´ë¥  ì»¤ë¸Œ.
ì‚¬ê±´(Event) + ì‹œê°„(Time) ê¸°ë°˜ìœ¼ë¡œ ê·¸ë˜í”„ë¥¼ ê·¸ë¦¬ëŠ”ë°â€¦
ì—¬ê¸°ì„œ Left Censored / Right Censored ë°ì´í„°, incomplete ë°ì´í„°ë¥¼ ê³ ë ¤í•´ì„œ Estimation í•˜ê¸°

ì‹œê°„ ìˆœìœ¼ë¡œ ì •ë ¬ í›„, ì‹œê°„ì˜ êµ¬ê°„ì— ëŒ€í•´ êµ¬ê°„ìƒì¡´ë¥  $P(t)$ ëŠ” $\cfrac{Survivors @ t}{Total Observation @ t}$
ê·¸ëŸ¼ ëˆ„ì ìƒì¡´ë¥  $S(t) = S(t-1) \times P(t)$ ë¥¼ y ì¶•ìœ¼ë¡œ, ì‹œê°„ì„ x ì¶•ìœ¼ë¡œ í•˜ëŠ” curveë¥¼ ê·¸ë¦´ ìˆ˜ ìˆìŒ.
ë¹„ìŠ·í•˜ê²Œ Nelson-Aalen estimatorëŠ” ìœ„í—˜ë„ë¥¼ cumulative í•˜ê²Œ ê³„ì‚°

ë‹¤ë§Œ Covariative ì˜í–¥ì´ ë‚¨ê¸° ë•Œë¬¸ì—â€¦ ë³´ì • í•´ì•¼ê² ì§€?
ì—¬ê¸°ì— Log-rank testë‚˜ Spearman Correl. êµ¬í•˜ë©´ ì¼ë°˜ì 

##### Cox Proportional Hazard Model
- ì¡°ê±´ : ìƒì¡´ í•¨ìˆ˜ $S(t)$ëŠ” Exponential í•˜ë©°, ë‘ êµ°ê°„ ìœ„í—˜ë¹„ëŠ” ëª¨ë“  ì‹œê°„ì— ëŒ€í•´ ì¼ì •í•¨

ìƒì¡´ í•¨ìˆ˜(Survival) $S(t)$ëŠ” $Prob(t)$ ë¡œ, t ì‹œì ì— ì‚´ì•„ìˆì„ í™•ë¥ ì„ ëœ»í•¨
ì´ë¥¼ í†µí•´ íŠ¹ì • ì‹œê°„ ì•ˆì— ì‚¬ê±´ì´ ë°œìƒí•  í™•ë¥  $F(t)$ ëŠ” $l-S(t)$ì´ë©°, ì´ë¥¼ ë¯¸ë¶„í•˜ë©´ ìƒì¡´ë°€ë„í•¨ìˆ˜ $f(t) = \cfrac{d}{dt}F(t)$

ë°˜ëŒ€ë¡œ ìœ„í—˜ í•¨ìˆ˜(Hazard) $h(t)$ëŠ” t ì‹œì ì— ì‚¬ê±´ì´ ë°œìƒí•˜ì§€ ì•Šì€ ì§‘ë‹¨ì— ëŒ€í•´ ì‚¬ê±´ì´ ë°œìƒí•  í™•ë¥ ì„ ëœ»í•¨. $h(t) = f(t) / S(t) = -\cfrac{d}{dt}logS(t)$
ì´ê±¸ ì ë¶„í•˜ë©´ ì‹œê°„ tê¹Œì§€ì˜ ì‚¬ê±´ ë°œìƒë¥ ì¸ ëˆ„ì ìœ„í—˜í•¨ìˆ˜ $H(t) = \int_{0}^{t}h(u)du$ê°€ ë¨

ì´ë“¤ì€ $S(t) = e^{-H(t)} = 1-F(t)$ ì´ë©°, ìœ„í—˜ë¥  $HR = h_2(t) / h_1(t)$ ëŠ” êµ° 1ê³¼ êµ° 2ê°„ì˜ ìœ„í—˜ í•¨ìˆ˜ì˜ ë¹„ë¥¼ ì˜ë¯¸í•¨

##### Sashimi plot

##### Rainfall plot



##### Sankey diagram
Flow the value to another value.
See mermaid diagram example for help :)

##### Electropherogram
ì „ê¸°ì˜ë™ë„ ê·¸ë˜í”„
í”íˆ sanger sequencingì´ë‚˜ QCí•  ë•Œ ë³´ëŠ” ê·¸ ê·¸ë¦¼

##### Rainfall plot
Somatic mutationì„ ê·¸ë¦¬ëŠ” scatter.

##### Karyotype plot

-

- 

- Re-squiggle : Basecall ì´í›„ signal ì •ë ¬ ë° dwell time normalize, ë‹¤ì‹œ Basecall

- 

- 
- 

- 
- CIGAR(


- DR(Discovery Rate)

-

-

-



- 

- 

- Analogue : subtitle bases

- 

- ONT(Oxford Nanopore Technologies) : ë°”ë¡œ ê·¸ íšŒì‚¬

- RBP(RNA Binding Protein)

- RNP(Ribonucleoprotein) complex : 

- NMD(Nonsense-mediated decay) : 

- NSD(Non-stop decay) : 

- SCR(Stop codon readthrough)

- PTC(Permature termation codon) <-> NTC(Normal...)

- 

- 

- 
- 
- 5' Flanking : 

	- Eukaryote : TATA, CAAT, GC...
	- Prokaryote : Consensus, A[CT]N{A}YR

Homopolymer : repaet of monomer

RNA Lifecycle : Transcription - 5'Cap - AtoI edit- Modifications - Splicing - 3'tail - expression @ mRNA

MPRA(Massively parallel report assay)

RRACH motif 
m6A consensus motif

Base modifications.. ([Wiki page][https://en.wikipedia.org/wiki/Nucleic_acid_analogue])



kozak consensus : GCCA/GCC**AUG**G, start codon AUG flanks





DEG(Differentially Expressed Genes) 

DET()

TSS(Transcription start site)

TES(Transciption end site)

APA(Alternative Polyadenlyation)

Alternative Exons

GENCODE annotation

Comprehensive annotation

MFE(Minimal Free Energy) + Structure

SRP(Single Recognition Particle)

IRES(Internal Ribosome Entry Site)

TIS(Total Isotropic Sensitivity)

PWM(Position weight matrix)

PSSM(Position specific scoring matrix)

BWT()

Sequence Logo

Circos plot

Volcano plot

RANSAC



MITEs(Miniature Inverted-repeat Transposable Elements)
